{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "── Attaching packages ─────────────────────────────────────── tidyverse 1.2.1 ──\n",
      "✔ ggplot2 3.1.0     ✔ purrr   0.2.5\n",
      "✔ tibble  1.4.2     ✔ dplyr   0.7.7\n",
      "✔ tidyr   0.8.2     ✔ stringr 1.3.1\n",
      "✔ readr   1.1.1     ✔ forcats 0.3.0\n",
      "── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──\n",
      "✖ dplyr::filter() masks stats::filter()\n",
      "✖ dplyr::lag()    masks stats::lag()\n",
      "\n",
      "Attaching package: ‘cowplot’\n",
      "\n",
      "The following object is masked from ‘package:ggplot2’:\n",
      "\n",
      "    ggsave\n",
      "\n",
      "Warning message:\n",
      "“replacing previous import ‘Biostrings::translate’ by ‘seqinr::translate’ when loading ‘dndscv’”Loading required package: stats4\n",
      "Loading required package: BiocGenerics\n",
      "Loading required package: parallel\n",
      "\n",
      "Attaching package: ‘BiocGenerics’\n",
      "\n",
      "The following objects are masked from ‘package:parallel’:\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, parApply, parCapply, parLapply,\n",
      "    parLapplyLB, parRapply, parSapply, parSapplyLB\n",
      "\n",
      "The following objects are masked from ‘package:dplyr’:\n",
      "\n",
      "    combine, intersect, setdiff, union\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    IQR, mad, sd, var, xtabs\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    anyDuplicated, append, as.data.frame, basename, cbind, colMeans,\n",
      "    colnames, colSums, dirname, do.call, duplicated, eval, evalq,\n",
      "    Filter, Find, get, grep, grepl, intersect, is.unsorted, lapply,\n",
      "    lengths, Map, mapply, match, mget, order, paste, pmax, pmax.int,\n",
      "    pmin, pmin.int, Position, rank, rbind, Reduce, rowMeans, rownames,\n",
      "    rowSums, sapply, setdiff, sort, table, tapply, union, unique,\n",
      "    unsplit, which, which.max, which.min\n",
      "\n",
      "Loading required package: S4Vectors\n",
      "\n",
      "Attaching package: ‘S4Vectors’\n",
      "\n",
      "The following objects are masked from ‘package:dplyr’:\n",
      "\n",
      "    first, rename\n",
      "\n",
      "The following object is masked from ‘package:tidyr’:\n",
      "\n",
      "    expand\n",
      "\n",
      "The following object is masked from ‘package:base’:\n",
      "\n",
      "    expand.grid\n",
      "\n",
      "Loading required package: IRanges\n",
      "\n",
      "Attaching package: ‘IRanges’\n",
      "\n",
      "The following objects are masked from ‘package:dplyr’:\n",
      "\n",
      "    collapse, desc, slice\n",
      "\n",
      "The following object is masked from ‘package:purrr’:\n",
      "\n",
      "    reduce\n",
      "\n",
      "Loading required package: GenomeInfoDb\n"
     ]
    }
   ],
   "source": [
    "library(tidyverse)\n",
    "library(readxl)\n",
    "library(cowplot)\n",
    "library(dndscv)\n",
    "library(GenomicRanges)\n",
    "library(readr)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1 Oesophagus data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this notebook, I'll use [dndsv](https://github.com/im3sanger/dndscv) to recalculate $dN/dS$ ratios from data in the paper [Somatic mutant clones colonize the human esophagus with age](https://www.ncbi.nlm.nih.gov/pubmed/30337457). \n",
    "\n",
    "I'll calculate $dN/dS$ across the frequency range by fixing a lower bound and increasing the upper bound step by step. This interval $dN/dS$ can then be used in our model framework to infer parameters related to the clonal dynamics."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Parsed with column specification:\n",
      "cols(\n",
      "  donor = col_character(),\n",
      "  chr = col_character(),\n",
      "  pos = col_integer(),\n",
      "  ref = col_character(),\n",
      "  mut = col_character(),\n",
      "  sumvaf = col_double(),\n",
      "  qval = col_double(),\n",
      "  gene = col_character(),\n",
      "  impact = col_character(),\n",
      "  aachange = col_character(),\n",
      "  ntchange = col_character(),\n",
      "  pid = col_character(),\n",
      "  merged = col_character()\n",
      ")\n"
     ]
    }
   ],
   "source": [
    "#read in donor info data\n",
    "dfdonor <- read_xlsx(\"data/esophagus/patient_info.xlsx\", skip = 1) %>%\n",
    "  dplyr::rename(patient = PD)\n",
    "\n",
    "#read in mutation data\n",
    "df <- read_csv(\"data/esophagus/esophagus.csv\") %>%\n",
    "  mutate(sumvaf = sumvaf * 2)\n",
    "\n",
    "target_genes <- unique(df$gene)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First I'll calculate interval $dN/dS$ ratios across the whole cohort"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 10.9).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3.”    Regression model for indels (theta = 2.97)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.470944 1.287522 1.680495\n",
      "wnon wnon 3.978046 3.224557 4.907605\n",
      "wspl wspl 3.418671 2.580541 4.529017\n",
      "wtru wtru 3.784218 3.148648 4.548081\n",
      "wall wall 1.669137 1.464509 1.902357\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.86).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1.”    Regression model for indels (theta = 4.78)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.724943 1.563642 1.902883\n",
      "wnon wnon 5.859105 5.094970 6.737844\n",
      "wspl wspl 5.860288 4.864851 7.059409\n",
      "wtru wtru 5.859472 5.169253 6.641851\n",
      "wall wall 2.071186 1.881189 2.280372\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.57).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, KMT2D.”    Regression model for indels (theta = 4.66)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.852630 1.685677 2.036119\n",
      "wnon wnon 6.443111 5.645040 7.354011\n",
      "wspl wspl 6.769507 5.691989 8.051004\n",
      "wtru wtru 6.543533 5.810112 7.369535\n",
      "wall wall 2.242892 2.044555 2.460470\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.2).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, KMT2D, ARID2.”    Regression model for indels (theta = 4.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.920141 1.750292 2.106472\n",
      "wnon wnon 6.724158 5.910632 7.649656\n",
      "wspl wspl 6.986024 5.902058 8.269071\n",
      "wtru wtru 6.805169 6.059304 7.642844\n",
      "wall wall 2.326564 2.124555 2.547780\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.19).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, KMT2D, ARID2.”    Regression model for indels (theta = 4.65)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.950722 1.779748 2.138122\n",
      "wnon wnon 6.969177 6.138675 7.912038\n",
      "wspl wspl 7.299769 6.184953 8.615526\n",
      "wtru wtru 7.070867 6.306481 7.927903\n",
      "wall wall 2.375490 2.171131 2.599085\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.07).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.77)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.980246 1.807662 2.169307\n",
      "wnon wnon 7.188643 6.339596 8.151401\n",
      "wspl wspl 7.475561 6.344012 8.808938\n",
      "wtru wtru 7.276991 6.496596 8.151130\n",
      "wall wall 2.417547 2.210722 2.643722\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.89).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.79)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.997570 1.824068 2.187576\n",
      "wnon wnon 7.277655 6.422615 8.246527\n",
      "wspl wspl 7.682506 6.530405 9.037863\n",
      "wtru wtru 7.402240 6.612627 8.286141\n",
      "wall wall 2.441639 2.233445 2.669240\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.95).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.79)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.989693 1.817667 2.177999\n",
      "wnon wnon 7.247570 6.399603 8.207895\n",
      "wspl wspl 7.843459 6.677138 9.213507\n",
      "wtru wtru 7.430433 6.641713 8.312816\n",
      "wall wall 2.434917 2.228273 2.660725\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.92).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.79)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.000184 1.827648 2.189008\n",
      "wnon wnon 7.358684 6.500705 8.329901\n",
      "wspl wspl 7.823266 6.662373 9.186441\n",
      "wtru wtru 7.501932 6.707996 8.389836\n",
      "wall wall 2.449897 2.242443 2.676544\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.8).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 5.04)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.001923 1.829527 2.190564\n",
      "wnon wnon 7.364752 6.507644 8.334749\n",
      "wspl wspl 7.828501 6.668836 9.189823\n",
      "wtru wtru 7.507682 6.714490 8.394575\n",
      "wall wall 2.451663 2.244400 2.678065\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.79).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, AJUBA, CCND1, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 5.04)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.011465 1.838328 2.200909\n",
      "wnon wnon 7.390940 6.531460 8.363519\n",
      "wspl wspl 7.917895 6.747945 9.290690\n",
      "wtru wtru 7.553247 6.755954 8.444631\n",
      "wall wall 2.463103 2.254953 2.690467\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.87).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 5.04)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.016654 1.843247 2.206375\n",
      "wnon wnon 7.412854 6.552361 8.386353\n",
      "wspl wspl 7.981515 6.804351 9.362330\n",
      "wtru wtru 7.587490 6.787735 8.481475\n",
      "wall wall 2.470311 2.261764 2.698088\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.86).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.025190 1.851121 2.215627\n",
      "wnon wnon 7.469794 6.603778 8.449380\n",
      "wspl wspl 8.066201 6.879389 9.457759\n",
      "wtru wtru 7.653010 6.847323 8.553497\n",
      "wall wall 2.482816 2.273298 2.711645\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.86).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.031141 1.856615 2.222074\n",
      "wnon wnon 7.534698 6.662115 8.521570\n",
      "wspl wspl 8.112558 6.920043 9.510578\n",
      "wtru wtru 7.712311 6.901082 8.618899\n",
      "wall wall 2.492317 2.282061 2.721944\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.033936 1.859203 2.225090\n",
      "wnon wnon 7.573929 6.697464 8.565093\n",
      "wspl wspl 8.122699 6.928745 9.522395\n",
      "wtru wtru 7.742602 6.928570 8.652274\n",
      "wall wall 2.496818 2.286219 2.726816\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.039941 1.864742 2.231601\n",
      "wnon wnon 7.616570 6.735786 8.612526\n",
      "wspl wspl 8.143514 6.946532 9.546752\n",
      "wtru wtru 7.778491 6.961044 8.691932\n",
      "wall wall 2.504891 2.293664 2.735569\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.040805 1.865539 2.232538\n",
      "wnon wnon 7.674456 6.788006 8.676667\n",
      "wspl wspl 8.176608 6.975767 9.584168\n",
      "wtru wtru 7.828818 7.006795 8.747280\n",
      "wall wall 2.509610 2.298010 2.740694\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.044145 1.868624 2.236154\n",
      "wnon wnon 7.669658 6.783875 8.671099\n",
      "wspl wspl 8.185597 6.983480 9.594644\n",
      "wtru wtru 7.828091 7.006198 8.746400\n",
      "wall wall 2.512487 2.300671 2.743805\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.048971 1.873067 2.241394\n",
      "wnon wnon 7.692220 6.804035 8.696347\n",
      "wspl wspl 8.220902 7.014503 9.634785\n",
      "wtru wtru 7.854599 7.030120 8.775771\n",
      "wall wall 2.518469 2.306181 2.750298\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.051828 1.875698 2.244496\n",
      "wnon wnon 7.709471 6.819504 8.715581\n",
      "wspl wspl 8.229048 7.021511 9.644253\n",
      "wtru wtru 7.869067 7.043200 8.791772\n",
      "wall wall 2.522138 2.309562 2.754280\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.053500 1.877240 2.246308\n",
      "wnon wnon 7.728295 6.836335 8.736633\n",
      "wspl wspl 8.238693 7.030913 9.653947\n",
      "wtru wtru 7.885355 7.057983 8.809716\n",
      "wall wall 2.524557 2.311791 2.756905\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.054429 1.878103 2.247309\n",
      "wnon wnon 7.735039 6.842539 8.743952\n",
      "wspl wspl 8.241097 7.033000 9.656716\n",
      "wtru wtru 7.890706 7.062902 8.815533\n",
      "wall wall 2.525718 2.312866 2.758160\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.054698 1.878354 2.247597\n",
      "wnon wnon 7.758942 6.864081 8.770463\n",
      "wspl wspl 8.243051 7.034686 9.658980\n",
      "wtru wtru 7.907892 7.078523 8.834436\n",
      "wall wall 2.527396 2.314412 2.759979\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.055736 1.879313 2.248722\n",
      "wnon wnon 7.788629 6.890766 8.803482\n",
      "wspl wspl 8.240360 7.032599 9.655538\n",
      "wtru wtru 7.927773 7.096617 8.856274\n",
      "wall wall 2.529944 2.316761 2.762743\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.058279 1.881660 2.251476\n",
      "wnon wnon 7.801833 6.902836 8.817912\n",
      "wspl wspl 8.269760 7.058955 9.688251\n",
      "wtru wtru 7.945971 7.113246 8.876180\n",
      "wall wall 2.533627 2.320163 2.766731\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.058821 1.882163 2.252060\n",
      "wnon wnon 7.810835 6.910981 8.827856\n",
      "wspl wspl 8.271775 7.060688 9.690594\n",
      "wtru wtru 7.952811 7.119474 8.883691\n",
      "wall wall 2.534615 2.321076 2.767799\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.93)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.058821 1.882163 2.252060\n",
      "wnon wnon 7.810835 6.910981 8.827856\n",
      "wspl wspl 8.271775 7.060688 9.690594\n",
      "wtru wtru 7.952811 7.119474 8.883691\n",
      "wall wall 2.534615 2.321076 2.767799\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.058821 1.882163 2.252060\n",
      "wnon wnon 7.810835 6.910981 8.827856\n",
      "wspl wspl 8.271775 7.060688 9.690594\n",
      "wtru wtru 7.952811 7.119474 8.883691\n",
      "wall wall 2.534615 2.321076 2.767799\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.059223 1.882535 2.252494\n",
      "wnon wnon 7.812074 6.912075 8.829259\n",
      "wspl wspl 8.273130 7.061847 9.692178\n",
      "wtru wtru 7.954087 7.120615 8.885117\n",
      "wall wall 2.535008 2.321440 2.768224\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.059223 1.882535 2.252494\n",
      "wnon wnon 7.812074 6.912075 8.829259\n",
      "wspl wspl 8.273130 7.061847 9.692178\n",
      "wtru wtru 7.954087 7.120615 8.885117\n",
      "wall wall 2.535008 2.321440 2.768224\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.059912 1.883170 2.253243\n",
      "wnon wnon 7.801929 6.903173 8.817698\n",
      "wspl wspl 8.274626 7.063124 9.693931\n",
      "wtru wtru 7.947398 7.114658 8.877607\n",
      "wall wall 2.535346 2.321753 2.768589\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.061365 1.884504 2.254825\n",
      "wnon wnon 7.815206 6.915140 8.832422\n",
      "wspl wspl 8.280076 7.067771 9.700323\n",
      "wtru wtru 7.958246 7.124495 8.889568\n",
      "wall wall 2.537629 2.323854 2.771071\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.062875 1.885897 2.256461\n",
      "wnon wnon 7.819852 6.919236 8.837694\n",
      "wspl wspl 8.284948 7.071938 9.706018\n",
      "wtru wtru 7.962970 7.128713 8.894858\n",
      "wall wall 2.539132 2.325240 2.772698\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.062875 1.885897 2.256461\n",
      "wnon wnon 7.819852 6.919236 8.837694\n",
      "wspl wspl 8.284948 7.071938 9.706018\n",
      "wtru wtru 7.962970 7.128713 8.894858\n",
      "wall wall 2.539132 2.325240 2.772698\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.063432 1.886412 2.257063\n",
      "wnon wnon 7.821562 6.920745 8.839632\n",
      "wspl wspl 8.286634 7.073385 9.707983\n",
      "wtru wtru 7.964676 7.130240 8.896766\n",
      "wall wall 2.539680 2.325747 2.773291\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.063432 1.886412 2.257063\n",
      "wnon wnon 7.821562 6.920745 8.839632\n",
      "wspl wspl 8.286634 7.073385 9.707983\n",
      "wtru wtru 7.964676 7.130240 8.896766\n",
      "wall wall 2.539680 2.325747 2.773291\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.063432 1.886412 2.257063\n",
      "wnon wnon 7.821562 6.920745 8.839632\n",
      "wspl wspl 8.286634 7.073385 9.707983\n",
      "wtru wtru 7.964676 7.130240 8.896766\n",
      "wall wall 2.539680 2.325747 2.773291\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.063432 1.886412 2.257063\n",
      "wnon wnon 7.821562 6.920745 8.839632\n",
      "wspl wspl 8.286634 7.073385 9.707983\n",
      "wtru wtru 7.964676 7.130240 8.896766\n",
      "wall wall 2.539680 2.325747 2.773291\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.065160 1.888007 2.258935\n",
      "wnon wnon 7.810498 6.911106 8.826934\n",
      "wspl wspl 8.286543 7.073432 9.707705\n",
      "wtru wtru 7.956861 7.123334 8.887921\n",
      "wall wall 2.540885 2.326862 2.774593\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.066547 1.889286 2.260439\n",
      "wnon wnon 7.809582 6.910369 8.825805\n",
      "wspl wspl 8.290874 7.077129 9.712779\n",
      "wtru wtru 7.957481 7.123922 8.888574\n",
      "wall wall 2.542120 2.328003 2.775931\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.066716 1.889443 2.260622\n",
      "wnon wnon 7.811057 6.911666 8.827483\n",
      "wspl wspl 8.320870 7.103777 9.746488\n",
      "wtru wtru 7.967751 7.133243 8.899887\n",
      "wall wall 2.542876 2.328700 2.776750\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.067081 1.889781 2.261016\n",
      "wnon wnon 7.812045 6.912539 8.828600\n",
      "wspl wspl 8.317943 7.101380 9.742920\n",
      "wtru wtru 7.967597 7.133125 8.899691\n",
      "wall wall 2.543198 2.328998 2.777097\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.066896 1.889612 2.260814\n",
      "wnon wnon 7.812067 6.912557 8.828628\n",
      "wspl wspl 8.335556 7.117272 9.762378\n",
      "wtru wtru 7.973100 7.138156 8.905705\n",
      "wall wall 2.543331 2.329122 2.777240\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.067122 1.889820 2.261058\n",
      "wnon wnon 7.827195 6.926158 8.845450\n",
      "wspl wspl 8.337313 7.118771 9.764437\n",
      "wtru wtru 7.984146 7.148175 8.917883\n",
      "wall wall 2.544352 2.330062 2.778349\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.070508 1.892940 2.264733\n",
      "wnon wnon 7.837604 6.935324 8.857268\n",
      "wspl wspl 8.348102 7.128002 9.777047\n",
      "wtru wtru 7.994692 7.157585 8.929703\n",
      "wall wall 2.547742 2.333188 2.782027\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.070508 1.892940 2.264733\n",
      "wnon wnon 7.837604 6.935324 8.857268\n",
      "wspl wspl 8.348102 7.128002 9.777047\n",
      "wtru wtru 7.994692 7.157585 8.929703\n",
      "wall wall 2.547742 2.333188 2.782027\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.84).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.070508 1.892940 2.264733\n",
      "wnon wnon 7.837604 6.935324 8.857268\n",
      "wspl wspl 8.348102 7.128002 9.777047\n",
      "wtru wtru 7.994692 7.157585 8.929703\n",
      "wall wall 2.547742 2.333188 2.782027\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.88).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.067494 1.890298 2.261302\n",
      "wnon wnon 7.822770 6.922619 8.839968\n",
      "wspl wspl 8.336897 7.118675 9.763593\n",
      "wtru wtru 7.980883 7.145629 8.913769\n",
      "wall wall 2.544025 2.329926 2.777799\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.88).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.067759 1.890543 2.261587\n",
      "wnon wnon 7.823581 6.923336 8.840886\n",
      "wspl wspl 8.337674 7.119345 9.764495\n",
      "wtru wtru 7.981686 7.146350 8.914665\n",
      "wall wall 2.544282 2.330164 2.778076\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.88).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, CUL3, CCND1, AJUBA, PIK3CA, ARID2, KMT2D.”    Regression model for indels (theta = 4.91)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 2.068378 1.891113 2.262258\n",
      "wnon wnon 7.813586 6.914561 8.829500\n",
      "wspl wspl 8.338719 7.120243 9.765710\n",
      "wtru wtru 7.974949 7.140345 8.907107\n",
      "wall wall 2.544508 2.330373 2.778319\n"
     ]
    }
   ],
   "source": [
    "minarea <- 0.02\n",
    "maxarea <- 2.5\n",
    "step <- 0.05\n",
    "areacutoff <- seq(minarea, maxarea, step)\n",
    "target_genes <- unique(df$gene)\n",
    "\n",
    "dfdnds <- data.frame()\n",
    "dfdnds.genes <- data.frame()\n",
    "\n",
    "for (cutoff in areacutoff){\n",
    "  print(cutoff)\n",
    "  x1 <- df %>%\n",
    "    filter(sumvaf < cutoff) #filter for mutations with sumvaf < cutoff\n",
    "  x <- dndscv(x1, gene_list = target_genes,\n",
    "              outp = 3, max_muts_per_gene_per_sample = Inf, \n",
    "              max_coding_muts_per_sample = Inf)\n",
    "  out <- x$globaldnds %>%\n",
    "    mutate(areacutoff = cutoff, nmutations = length(x1$donor))\n",
    "  dfdnds <- rbind(dfdnds, out)\n",
    "  combined <- left_join(x$sel_cv, x$sel_loc, \n",
    "                        by = c(\"gene_name\", \"n_syn\", \"n_mis\", \"n_non\", \"n_spl\"))\n",
    "  out2 <- combined %>%\n",
    "    mutate(areacutoff = cutoff)\n",
    "  dfdnds.genes <- rbind(dfdnds.genes, out2)\n",
    "  print(x$globaldnds)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#write results to file\n",
    "write_csv(dfdnds, \"data/esophagus/dnds_combined.csv\")\n",
    "write_csv(dfdnds.genes, \"data/esophagus/dnds_genes_combined.csv\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now I'll calculate interval $dN/dS$ on a donor by donor basis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] \"PD36806\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.24).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1.”    Regression model for indels (theta = 3.31)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.050626 0.5329708  2.071060\n",
      "wnon wnon 2.847422 0.9424148  8.603231\n",
      "wspl wspl 4.838163 1.4991419 15.614149\n",
      "wtru wtru 3.578295 1.4530270  8.812082\n",
      "wall wall 1.262727 0.6514573  2.447558\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.563).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.338241 0.8164619  2.193475\n",
      "wnon wnon 5.355054 2.6608848 10.777095\n",
      "wspl wspl 7.563687 3.2734387 17.476837\n",
      "wtru wtru 6.060694 3.2815445 11.193511\n",
      "wall wall 1.694659 1.0454558  2.747003\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.668).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.507182 0.9322923  2.436572\n",
      "wnon wnon 5.748858 2.9325995 11.269650\n",
      "wspl wspl 7.657939 3.3622418 17.441942\n",
      "wtru wtru 6.348732 3.5017275 11.510433\n",
      "wall wall 1.887950 1.1801753  3.020192\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.662).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.573016 0.975853  2.535608\n",
      "wnon wnon 5.929997 3.056273 11.505803\n",
      "wspl wspl 7.900075 3.471137 17.980042\n",
      "wtru wtru 6.529371 3.620603 11.775025\n",
      "wall wall 1.965717 1.231517  3.137629\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.656).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.645237 1.022620  2.646931\n",
      "wnon wnon 6.560050 3.424889 12.565154\n",
      "wspl wspl 8.204425 3.605878 18.667458\n",
      "wtru wtru 7.053481 3.937503 12.635317\n",
      "wall wall 2.078577 1.304583  3.311771\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.657).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3.”    Regression model for indels (theta = 5.74)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.687212 1.049912  2.711356\n",
      "wnon wnon 6.554195 3.428977 12.527781\n",
      "wspl wspl 8.353603 3.672122 19.003368\n",
      "wtru wtru 7.085544 3.960087 12.677734\n",
      "wall wall 2.120286 1.331941  3.375233\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.658).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH3, TP53.”    Regression model for indels (theta = 5.45)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.700159 1.058431  2.730966\n",
      "wnon wnon 6.471969 3.391553 12.350206\n",
      "wspl wspl 8.393351 3.689909 19.092159\n",
      "wtru wtru 7.031213 3.933145 12.569575\n",
      "wall wall 2.129808 1.338325  3.389373\n",
      "[1] \"PD36712\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "Warning message:\n",
      "“glm.fit: fitted rates numerically 0 occurred”[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: .”    Regression model for indels (theta = 1.09)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name          mle     cilow   cihigh\n",
      "wmis wmis 1.417818e+00 0.8851876 2.270938\n",
      "wnon wnon 3.655357e+00 1.6737213 7.983190\n",
      "wspl wspl 9.552197e-08 0.0000000      Inf\n",
      "wtru wtru 2.328081e+00 1.1188136 4.844384\n",
      "wall wall 1.519629e+00 0.9558281 2.415992\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, FAT1, NOTCH2.”    Regression model for indels (theta = 4.62)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.869172 1.300958  2.685562\n",
      "wnon wnon 6.602098 3.967293 10.986758\n",
      "wspl wspl 4.177872 1.879576  9.286466\n",
      "wtru wtru 5.880244 3.687798  9.376128\n",
      "wall wall 2.219261 1.555590  3.166077\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.16).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.961989 1.375427  2.798695\n",
      "wnon wnon 7.261914 4.427535 11.910778\n",
      "wspl wspl 5.040795 2.408682 10.549180\n",
      "wtru wtru 6.584775 4.192662 10.341703\n",
      "wall wall 2.353495 1.661066  3.334570\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.31).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.972198 1.388952  2.800361\n",
      "wnon wnon 6.969484 4.271838 11.370679\n",
      "wspl wspl 4.991828 2.391800 10.418239\n",
      "wtru wtru 6.382986 4.079450  9.987255\n",
      "wall wall 2.346523 1.663253  3.310485\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.31).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.025269 1.427810  2.872733\n",
      "wnon wnon 6.775191 4.157916 11.039956\n",
      "wspl wspl 5.050598 2.422041 10.531836\n",
      "wtru wtru 6.275953 4.012881  9.815289\n",
      "wall wall 2.387404 1.693429  3.365773\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.047109 1.443976  2.902164\n",
      "wnon wnon 6.833854 4.193653 11.136248\n",
      "wspl wspl 5.089499 2.441193 10.610797\n",
      "wtru wtru 6.328215 4.046304  9.897009\n",
      "wall wall 2.408830 1.709285  3.394672\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.053444 1.448757  2.910519\n",
      "wnon wnon 6.857945 4.208314 11.175830\n",
      "wspl wspl 5.431060 2.655640 11.107083\n",
      "wtru wtru 6.439565 4.123062 10.057575\n",
      "wall wall 2.420637 1.718007  3.410628\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.053444 1.448757  2.910519\n",
      "wnon wnon 6.857945 4.208314 11.175830\n",
      "wspl wspl 5.431060 2.655640 11.107083\n",
      "wtru wtru 6.439565 4.123062 10.057575\n",
      "wall wall 2.420637 1.718007  3.410628\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.062781 1.455549  2.923340\n",
      "wnon wnon 6.721264 4.127222 10.945716\n",
      "wspl wspl 5.444069 2.662152 11.133055\n",
      "wtru wtru 6.352263 4.067906  9.919415\n",
      "wall wall 2.424064 1.720567  3.415204\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.062781 1.455549  2.923340\n",
      "wnon wnon 6.721264 4.127222 10.945716\n",
      "wspl wspl 5.444069 2.662152 11.133055\n",
      "wtru wtru 6.352263 4.067906  9.919415\n",
      "wall wall 2.424064 1.720567  3.415204\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.3).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH3, NOTCH1, NOTCH2, FAT1, TP53.”    Regression model for indels (theta = 6.1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.074326 1.464013  2.939062\n",
      "wnon wnon 6.751668 4.145650 10.995870\n",
      "wspl wspl 5.469462 2.674611 11.184809\n",
      "wtru wtru 6.381159 4.086231  9.964975\n",
      "wall wall 2.435522 1.728974  3.430802\n",
      "[1] \"PD30272\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.18e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle      cilow   cihigh\n",
      "wmis wmis 1.6141468 1.04371348 2.496346\n",
      "wnon wnon 1.4093173 0.62869106 3.159223\n",
      "wspl wspl 0.4808522 0.06265063 3.690606\n",
      "wtru wtru 1.1682718 0.54727892 2.493900\n",
      "wall wall 1.5589859 1.01413102 2.396571\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53.”    Regression model for indels (theta = 2.39)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.400240 1.025049 1.912761\n",
      "wnon wnon 2.270931 1.365819 3.775849\n",
      "wspl wspl 2.419212 1.111105 5.267355\n",
      "wtru wtru 2.308495 1.462838 3.643022\n",
      "wall wall 1.496021 1.101409 2.032014\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.78e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.391202 1.024526 1.889111\n",
      "wnon wnon 2.451005 1.499936 4.005121\n",
      "wspl wspl 2.542732 1.207460 5.354619\n",
      "wtru wtru 2.474793 1.592229 3.846557\n",
      "wall wall 1.503023 1.113118 2.029503\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.79e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.398119 1.029872 1.898038\n",
      "wnon wnon 2.459963 1.505366 4.019897\n",
      "wspl wspl 2.550969 1.211502 5.371386\n",
      "wtru wtru 2.483579 1.597872 3.860237\n",
      "wall wall 1.509771 1.118335 2.038215\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.73e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.416450 1.043880 1.921995\n",
      "wnon wnon 2.465452 1.509546 4.026675\n",
      "wspl wspl 2.576763 1.223746 5.425723\n",
      "wtru wtru 2.494142 1.605165 3.875452\n",
      "wall wall 1.527066 1.131602 2.060734\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.424973 1.050424 1.933075\n",
      "wnon wnon 2.476414 1.516173 4.044807\n",
      "wspl wspl 2.588096 1.229200 5.449265\n",
      "wtru wtru 2.505209 1.612232 3.892785\n",
      "wall wall 1.535426 1.138024 2.071603\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 7.67e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, NOTCH3, TP53, CUL3.”    Regression model for indels (theta = 3.44)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.435132 1.058150 1.946419\n",
      "wnon wnon 2.471295 1.513924 4.034086\n",
      "wspl wspl 2.602137 1.235847 5.478929\n",
      "wtru wtru 2.504803 1.612525 3.890817\n",
      "wall wall 1.544693 1.145117 2.083697\n",
      "[1] \"PD30986\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 6.99e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3.”    Regression model for indels (theta = 0.81)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.354622 0.9560226  1.919413\n",
      "wnon wnon 3.905420 2.2660891  6.730674\n",
      "wspl wspl 5.530368 2.8297581 10.808335\n",
      "wtru wtru 4.413244 2.7586786  7.060163\n",
      "wall wall 1.583103 1.1248418  2.228061\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 16.5).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH2, NOTCH3, CUL3, FAT1.”    Regression model for indels (theta = 1.17)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.817491 1.398308  2.362335\n",
      "wnon wnon 5.718624 3.890066  8.406710\n",
      "wspl wspl 7.614823 4.737462 12.239788\n",
      "wtru wtru 6.312811 4.497696  8.860444\n",
      "wall wall 2.145075 1.657589  2.775927\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.07e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3.”    Regression model for indels (theta = 0.769)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.172582 1.685940  2.799692\n",
      "wnon wnon  6.602905 4.605546  9.466489\n",
      "wspl wspl 10.143475 6.570869 15.658520\n",
      "wtru wtru  7.665047 5.572426 10.543513\n",
      "wall wall  2.582409 2.012208  3.314189\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 2.47e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, FAT1, NOTCH3, CUL3, SPHKAP.”    Regression model for indels (theta = 0.676)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.332389 1.818517  2.991470\n",
      "wnon wnon  7.079792 4.984327 10.056215\n",
      "wspl wspl 10.347569 6.751297 15.859499\n",
      "wtru wtru  8.058890 5.897460 11.012488\n",
      "wall wall  2.757697 2.158283  3.523584\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.41e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.676)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.270486 1.783176  2.890970\n",
      "wnon wnon  7.233935 5.151501 10.158169\n",
      "wspl wspl 10.076324 6.618875 15.339813\n",
      "wtru wtru  8.077975 5.965268 10.938936\n",
      "wall wall  2.704083 2.131975  3.429714\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.45e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.676)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.316672 1.820250  2.948480\n",
      "wnon wnon  7.706337 5.514424 10.769506\n",
      "wspl wspl 10.263351 6.742488 15.622775\n",
      "wtru wtru  8.457494 6.261238 11.424129\n",
      "wall wall  2.780618 2.193401  3.525044\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.49e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.595)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.366853 1.860579  3.010887\n",
      "wnon wnon  7.799295 5.590761 10.880272\n",
      "wspl wspl 10.335076 6.812027 15.680178\n",
      "wtru wtru  8.548142 6.337444 11.530001\n",
      "wall wall  2.836313 2.238221  3.594228\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 2.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.595)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.347553 1.847857  2.982377\n",
      "wnon wnon  7.776699 5.583801 10.830803\n",
      "wspl wspl 10.544204 6.983355 15.920749\n",
      "wtru wtru  8.596225 6.385310 11.572671\n",
      "wall wall  2.818949 2.227538  3.567379\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 2.56e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.382463 1.875904  3.025810\n",
      "wnon wnon  7.933626 5.699583 11.043338\n",
      "wspl wspl 10.794691 7.167025 16.258538\n",
      "wtru wtru  8.786635 6.531231 11.820888\n",
      "wall wall  2.863458 2.263322  3.622724\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 2.57e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.392285 1.883830  3.037976\n",
      "wnon wnon  7.960602 5.718798 11.081209\n",
      "wspl wspl 10.825003 7.187671 16.303011\n",
      "wtru wtru  8.815260 6.552454 11.859496\n",
      "wall wall  2.873179 2.271167  3.634764\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 2.55e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.399611 1.889757  3.047023\n",
      "wnon wnon  7.956256 5.716617 11.073335\n",
      "wspl wspl 10.847260 7.202764 16.335819\n",
      "wtru wtru  8.816840 6.554212 11.860567\n",
      "wall wall  2.879701 2.276453  3.642806\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.374789 1.872435  3.011919\n",
      "wnon wnon  7.877874 5.671052 10.943453\n",
      "wspl wspl 10.747097 7.141088 16.174019\n",
      "wtru wtru  8.725493 6.496048 11.720084\n",
      "wall wall  2.854818 2.259597  3.606833\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.38e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.390771 1.885329  3.031718\n",
      "wnon wnon  8.148080 5.876861 11.297052\n",
      "wspl wspl 10.817025 7.188087 16.278047\n",
      "wtru wtru  8.936264 6.659887 11.990714\n",
      "wall wall  2.887573 2.285962  3.647515\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397845 1.891097  3.040385\n",
      "wnon wnon  8.369366 6.044221 11.588968\n",
      "wspl wspl 10.854087 7.213067 16.333025\n",
      "wtru wtru  9.104407 6.789966 12.207753\n",
      "wall wall  2.905262 2.300226  3.669442\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.398890 1.892009  3.041568\n",
      "wnon wnon  8.458794 6.112945 11.704863\n",
      "wspl wspl 10.854374 7.213915 16.331971\n",
      "wtru wtru  9.167608 6.839609 12.287987\n",
      "wall wall  2.910390 2.304395  3.675745\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.39e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.406472 1.898084  3.051027\n",
      "wnon wnon  8.537870 6.172874 11.808960\n",
      "wspl wspl 10.887255 7.235720 16.381552\n",
      "wtru wtru  9.232646 6.889591 12.372541\n",
      "wall wall  2.921397 2.313229  3.689457\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.39e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.406472 1.898084  3.051027\n",
      "wnon wnon  8.537870 6.172874 11.808960\n",
      "wspl wspl 10.887255 7.235720 16.381552\n",
      "wtru wtru  9.232646 6.889591 12.372541\n",
      "wall wall  2.921397 2.313229  3.689457\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.417087 1.906621  3.064222\n",
      "wnon wnon  8.479182 6.131453 11.725857\n",
      "wspl wspl 10.921903 7.258739 16.433702\n",
      "wtru wtru  9.197186 6.863535 12.324295\n",
      "wall wall  2.929371 2.319676  3.699315\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.416309 1.906011  3.063229\n",
      "wnon wnon  8.471866 6.126513 11.715066\n",
      "wspl wspl 11.069855 7.370939 16.624976\n",
      "wtru wtru  9.236799 6.894818 12.374286\n",
      "wall wall  2.930823 2.320860  3.701094\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.38e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.423329 1.911651  3.071964\n",
      "wnon wnon  8.556927 6.191797 11.825485\n",
      "wspl wspl 11.089071 7.384437 16.652251\n",
      "wtru wtru  9.302393 6.946179 12.457859\n",
      "wall wall  2.943582 2.331113  3.716970\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.38e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.423329 1.911651  3.071964\n",
      "wnon wnon  8.556927 6.191797 11.825485\n",
      "wspl wspl 11.089071 7.384437 16.652251\n",
      "wtru wtru  9.302393 6.946179 12.457859\n",
      "wall wall  2.943582 2.331113  3.716970\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.38e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.423329 1.911651  3.071964\n",
      "wnon wnon  8.556927 6.191797 11.825485\n",
      "wspl wspl 11.089071 7.384437 16.652251\n",
      "wtru wtru  9.302393 6.946179 12.457859\n",
      "wall wall  2.943582 2.331113  3.716970\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.38e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.423329 1.911651  3.071964\n",
      "wnon wnon  8.556927 6.191797 11.825485\n",
      "wspl wspl 11.089071 7.384437 16.652251\n",
      "wtru wtru  9.302393 6.946179 12.457859\n",
      "wall wall  2.943582 2.331113  3.716970\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.37e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.422142 1.910745  3.070411\n",
      "wnon wnon  8.669009 6.276028 11.974408\n",
      "wspl wspl 11.089214 7.384805 16.651850\n",
      "wtru wtru  9.383293 7.008528 12.562721\n",
      "wall wall  2.947643 2.334401  3.721982\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.430655 1.917541  3.081073\n",
      "wnon wnon  8.694335 6.294063 12.009963\n",
      "wspl wspl 11.121909 7.406517 16.701083\n",
      "wtru wtru  9.410922 7.028866 12.600248\n",
      "wall wall  2.956255 2.341287  3.732750\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.429379 1.916533  3.079458\n",
      "wnon wnon  8.694154 6.293893 12.009787\n",
      "wspl wspl 11.261287 7.512504 16.880736\n",
      "wtru wtru  9.454187 7.062914 12.655068\n",
      "wall wall  2.957192 2.342053  3.733897\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.429379 1.916533  3.079458\n",
      "wnon wnon  8.694154 6.293893 12.009787\n",
      "wspl wspl 11.261287 7.512504 16.880736\n",
      "wtru wtru  9.454187 7.062914 12.655068\n",
      "wall wall  2.957192 2.342053  3.733897\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.439555 1.924747  3.092056\n",
      "wnon wnon  8.724359 6.315554 12.051902\n",
      "wspl wspl 11.297757 7.537109 16.934785\n",
      "wtru wtru  9.486601 7.086999 12.698689\n",
      "wall wall  2.967286 2.350204  3.746393\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.439555 1.924747  3.092056\n",
      "wnon wnon  8.724359 6.315554 12.051902\n",
      "wspl wspl 11.297757 7.537109 16.934785\n",
      "wtru wtru  9.486601 7.086999 12.698689\n",
      "wall wall  2.967286 2.350204  3.746393\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.439555 1.924747  3.092056\n",
      "wnon wnon  8.724359 6.315554 12.051902\n",
      "wspl wspl 11.297757 7.537109 16.934785\n",
      "wtru wtru  9.486601 7.086999 12.698689\n",
      "wall wall  2.967286 2.350204  3.746393\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.439555 1.924747  3.092056\n",
      "wnon wnon  8.724359 6.315554 12.051902\n",
      "wspl wspl 11.297757 7.537109 16.934785\n",
      "wtru wtru  9.486601 7.086999 12.698689\n",
      "wall wall  2.967286 2.350204  3.746393\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.439555 1.924747  3.092056\n",
      "wnon wnon  8.724359 6.315554 12.051902\n",
      "wspl wspl 11.297757 7.537109 16.934785\n",
      "wtru wtru  9.486601 7.086999 12.698689\n",
      "wall wall  2.967286 2.350204  3.746393\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 4.4e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, CUL3, SPHKAP.”    Regression model for indels (theta = 0.526)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.439555 1.924747  3.092056\n",
      "wnon wnon  8.724359 6.315554 12.051902\n",
      "wspl wspl 11.297757 7.537109 16.934785\n",
      "wtru wtru  9.486601 7.086999 12.698689\n",
      "wall wall  2.967286 2.350204  3.746393\n",
      "[1] \"PD30987\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 5.97).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, NOTCH3, NOTCH2.”    Regression model for indels (theta = 0.703)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.619337 1.090208  2.405277\n",
      "wnon wnon 7.130720 3.997165 12.720806\n",
      "wspl wspl 4.720914 2.128289 10.471807\n",
      "wtru wtru 6.259545 3.741172 10.473161\n",
      "wall wall 1.975793 1.340577  2.911997\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 29.5).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2, CUL3.”    Regression model for indels (theta = 1.2)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.182723 1.603563  2.971059\n",
      "wnon wnon 9.995380 6.679176 14.958076\n",
      "wspl wspl 8.264267 4.835875 14.123216\n",
      "wtru wtru 9.443131 6.530524 13.654757\n",
      "wall wall 2.806394 2.074599  3.796324\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 9.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, NOTCH3, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.697)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.255284 1.673314  3.039661\n",
      "wnon wnon 10.386666 7.069533 15.260250\n",
      "wspl wspl  9.654405 5.864373 15.893863\n",
      "wtru wtru 10.157919 7.137414 14.456681\n",
      "wall wall  2.932008 2.188743  3.927675\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 8.72).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, NOTCH3, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.697)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.303414 1.714511  3.094595\n",
      "wnon wnon 10.773176 7.369909 15.748001\n",
      "wspl wspl  9.974440 6.144191 16.192440\n",
      "wtru wtru 10.516908 7.427817 14.890695\n",
      "wall wall  3.007050 2.251801  4.015607\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, NOTCH3, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.270759 1.694330  3.043296\n",
      "wnon wnon 10.824402 7.428101 15.773572\n",
      "wspl wspl 10.093832 6.247297 16.308724\n",
      "wtru wtru 10.589803 7.502201 14.948138\n",
      "wall wall  2.978317 2.235849  3.967340\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, NOTCH3, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.322388 1.733815  3.110762\n",
      "wnon wnon 11.210970 7.710139 16.301373\n",
      "wspl wspl 10.049082 6.223584 16.226029\n",
      "wtru wtru 10.836595 7.686889 15.276894\n",
      "wall wall  3.048037 2.289269  4.058296\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.54).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.334311 1.742810  3.126565\n",
      "wnon wnon 11.431991 7.864312 16.618162\n",
      "wspl wspl 10.334302 6.421261 16.631904\n",
      "wtru wtru 11.075666 7.860667 15.605595\n",
      "wall wall  3.072741 2.308013  4.090852\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.51).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.344203 1.750415  3.139420\n",
      "wnon wnon 11.627504 8.003299 16.892890\n",
      "wspl wspl 10.379596 6.449649 16.704169\n",
      "wtru wtru 11.221854 7.967120 15.806213\n",
      "wall wall  3.091097 2.322086  4.114782\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.51).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.344203 1.750415  3.139420\n",
      "wnon wnon 11.627504 8.003299 16.892890\n",
      "wspl wspl 10.379596 6.449649 16.704169\n",
      "wtru wtru 11.221854 7.967120 15.806213\n",
      "wall wall  3.091097 2.322086  4.114782\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.362299 1.764278  3.163028\n",
      "wnon wnon 11.791047 8.120606 17.120493\n",
      "wspl wspl 10.450432 6.494047 16.817175\n",
      "wtru wtru 11.355616 8.064999 15.988846\n",
      "wall wall  3.118693 2.343220  4.150802\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.49).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.384296 1.781198  3.191597\n",
      "wnon wnon 11.881771 8.182362 17.253757\n",
      "wspl wspl 10.525200 6.541358 16.935295\n",
      "wtru wtru 11.440577 8.125048 16.109051\n",
      "wall wall  3.140526 2.360028  4.179146\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.49).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.384296 1.781198  3.191597\n",
      "wnon wnon 11.881771 8.182362 17.253757\n",
      "wspl wspl 10.525200 6.541358 16.935295\n",
      "wtru wtru 11.440577 8.125048 16.109051\n",
      "wall wall  3.140526 2.360028  4.179146\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.49).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.384296 1.781198  3.191597\n",
      "wnon wnon 11.881771 8.182362 17.253757\n",
      "wspl wspl 10.525200 6.541358 16.935295\n",
      "wtru wtru 11.440577 8.125048 16.109051\n",
      "wall wall  3.140526 2.360028  4.179146\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.49).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.384296 1.781198  3.191597\n",
      "wnon wnon 11.881771 8.182362 17.253757\n",
      "wspl wspl 10.525200 6.541358 16.935295\n",
      "wtru wtru 11.440577 8.125048 16.109051\n",
      "wall wall  3.140526 2.360028  4.179146\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.49).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.384296 1.781198  3.191597\n",
      "wnon wnon 11.881771 8.182362 17.253757\n",
      "wspl wspl 10.525200 6.541358 16.935295\n",
      "wtru wtru 11.440577 8.125048 16.109051\n",
      "wall wall  3.140526 2.360028  4.179146\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH3, FAT1, NOTCH2, AJUBA, ARID1A, NFE2L2.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.397337 1.791220  3.208552\n",
      "wnon wnon 11.935756 8.219078 17.333119\n",
      "wspl wspl 10.574141 6.571934 17.013632\n",
      "wtru wtru 11.492757 8.161813 16.183103\n",
      "wall wall  3.153511 2.370014  4.196021\n",
      "[1] \"PD30274\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 3.65e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1.”    Regression model for indels (theta = 12)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.4201581 0.9369632 2.152538\n",
      "wnon wnon 3.7852724 1.8435159 7.772261\n",
      "wspl wspl 0.8541545 0.2478360 2.943801\n",
      "wtru wtru 2.3562469 1.2539990 4.427355\n",
      "wall wall 1.5279209 1.0158014 2.298227\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 8.2e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.84e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.527253 1.128160 2.067528\n",
      "wnon wnon 4.811856 3.030350 7.640688\n",
      "wspl wspl 2.326112 1.212256 4.463410\n",
      "wtru wtru 3.768026 2.511849 5.652416\n",
      "wall wall 1.757713 1.307419 2.363096\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.57e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.89e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.626431 1.216524 2.174456\n",
      "wnon wnon 4.723902 3.036301 7.349484\n",
      "wspl wspl 2.617843 1.416813 4.836985\n",
      "wtru wtru 3.879175 2.629583 5.722579\n",
      "wall wall 1.854071 1.395241 2.463789\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.24e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.89e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.560565 1.183516 2.057736\n",
      "wnon wnon 4.630402 3.042675 7.046634\n",
      "wspl wspl 2.719378 1.522179 4.858179\n",
      "wtru wtru 3.878904 2.681554 5.610888\n",
      "wall wall 1.795822 1.370381 2.353343\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.41e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.602832 1.219402 2.106828\n",
      "wnon wnon 4.827908 3.208951 7.263651\n",
      "wspl wspl 3.089274 1.770441 5.390531\n",
      "wtru wtru 4.162368 2.904544 5.964898\n",
      "wall wall 1.860456 1.424093 2.430526\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.42e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.639670 1.250763 2.149501\n",
      "wnon wnon 4.960836 3.307496 7.440642\n",
      "wspl wspl 3.252621 1.886306 5.608606\n",
      "wtru wtru 4.300717 3.012686 6.139427\n",
      "wall wall 1.904206 1.461267 2.481408\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.644274 1.256957 2.150938\n",
      "wnon wnon 5.082455 3.413435 7.567552\n",
      "wspl wspl 3.541591 2.091215 5.997887\n",
      "wtru wtru 4.493098 3.167492 6.373474\n",
      "wall wall 1.927140 1.482267 2.505531\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.672482 1.279257 2.186578\n",
      "wnon wnon 5.043028 3.388909 7.504519\n",
      "wspl wspl 3.936722 2.359632 6.567880\n",
      "wtru wtru 4.624278 3.266859 6.545721\n",
      "wall wall 1.962198 1.509967 2.549871\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.41e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 3.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.692159 1.294932 2.211238\n",
      "wnon wnon 5.377183 3.628633 7.968319\n",
      "wspl wspl 3.887954 2.331246 6.484165\n",
      "wtru wtru 4.805076 3.403096 6.784633\n",
      "wall wall 1.997178 1.537651 2.594035\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.707639 1.307119 2.230883\n",
      "wnon wnon 5.373935 3.627685 7.960774\n",
      "wspl wspl 4.050803 2.446594 6.706877\n",
      "wtru wtru 4.864945 3.449145 6.861902\n",
      "wall wall 2.016267 1.552692 2.618248\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.718299 1.315589 2.244282\n",
      "wnon wnon 5.380274 3.632823 7.968279\n",
      "wspl wspl 4.067531 2.456834 6.734200\n",
      "wtru wtru 4.876372 3.457547 6.877422\n",
      "wall wall 2.026095 1.560512 2.630586\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.730348 1.325335 2.259131\n",
      "wnon wnon 5.592997 3.786307 8.261775\n",
      "wspl wspl 4.093281 2.472706 6.775958\n",
      "wtru wtru 5.016226 3.561987 7.064182\n",
      "wall wall 2.047364 1.577453 2.657258\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.759368 1.348352 2.295673\n",
      "wnon wnon 5.574674 3.777540 8.226780\n",
      "wspl wspl 4.266734 2.592464 7.022285\n",
      "wtru wtru 5.075218 3.607608 7.139865\n",
      "wall wall 2.076616 1.600653 2.694110\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.764875 1.352752 2.302555\n",
      "wnon wnon 5.638244 3.824799 8.311495\n",
      "wspl wspl 4.277488 2.599125 7.039642\n",
      "wtru wtru 5.119334 3.641238 7.197438\n",
      "wall wall 2.086097 1.608205 2.705999\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.772519 1.358901 2.312033\n",
      "wnon wnon 5.726173 3.888982 8.431269\n",
      "wspl wspl 4.294325 2.609431 7.067147\n",
      "wtru wtru 5.180566 3.687153 7.278856\n",
      "wall wall 2.097409 1.617219 2.720177\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.57e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.784814 1.368652 2.327518\n",
      "wnon wnon 5.757081 3.909736 8.477292\n",
      "wspl wspl 4.318034 2.623834 7.106171\n",
      "wtru wtru 5.208640 3.706967 7.318633\n",
      "wall wall 2.109529 1.626834 2.735444\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.780262 1.365119 2.321653\n",
      "wnon wnon 5.752359 3.906543 8.470310\n",
      "wspl wspl 4.482694 2.739510 7.335088\n",
      "wtru wtru 5.268237 3.751980 7.397247\n",
      "wall wall 2.109529 1.626834 2.735444\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.780262 1.365119 2.321653\n",
      "wnon wnon 5.752359 3.906543 8.470310\n",
      "wspl wspl 4.482694 2.739510 7.335088\n",
      "wtru wtru 5.268237 3.751980 7.397247\n",
      "wall wall 2.109529 1.626834 2.735444\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.780262 1.365119 2.321653\n",
      "wnon wnon 5.752359 3.906543 8.470310\n",
      "wspl wspl 4.482694 2.739510 7.335088\n",
      "wtru wtru 5.268237 3.751980 7.397247\n",
      "wall wall 2.109529 1.626834 2.735444\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.780262 1.365119 2.321653\n",
      "wnon wnon 5.752359 3.906543 8.470310\n",
      "wspl wspl 4.482694 2.739510 7.335088\n",
      "wtru wtru 5.268237 3.751980 7.397247\n",
      "wall wall 2.109529 1.626834 2.735444\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.783165 1.367506 2.325167\n",
      "wnon wnon 5.752962 3.907514 8.469981\n",
      "wspl wspl 4.618322 2.836977 7.518178\n",
      "wtru wtru 5.319498 3.790957 7.464358\n",
      "wall wall 2.115690 1.631761 2.743135\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.785020 1.369067 2.327348\n",
      "wnon wnon 5.826675 3.962202 8.568505\n",
      "wspl wspl 4.624114 2.840642 7.527323\n",
      "wtru wtru 5.367617 3.827651 7.527152\n",
      "wall wall 2.121253 1.636220 2.750066\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.779209 1.364551 2.319873\n",
      "wnon wnon 5.943306 4.046009 8.730303\n",
      "wspl wspl 4.616911 2.836302 7.515373\n",
      "wtru wtru 5.435112 3.878245 7.616959\n",
      "wall wall 2.121253 1.636220 2.750066\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.779209 1.364551 2.319873\n",
      "wnon wnon 5.943306 4.046009 8.730303\n",
      "wspl wspl 4.616911 2.836302 7.515373\n",
      "wtru wtru 5.435112 3.878245 7.616959\n",
      "wall wall 2.121253 1.636220 2.750066\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.783426 1.367972 2.325052\n",
      "wnon wnon 5.913952 4.027647 8.683688\n",
      "wspl wspl 4.825483 2.979470 7.815245\n",
      "wtru wtru 5.500089 3.927223 7.702892\n",
      "wall wall 2.128979 1.642380 2.759746\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.789031 1.372426 2.332098\n",
      "wnon wnon 5.928437 4.037413 8.705168\n",
      "wspl wspl 4.836457 2.986319 7.832824\n",
      "wtru wtru 5.513128 3.936494 7.721233\n",
      "wall wall 2.134485 1.646756 2.766668\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.789031 1.372426 2.332098\n",
      "wnon wnon 5.928437 4.037413 8.705168\n",
      "wspl wspl 4.836457 2.986319 7.832824\n",
      "wtru wtru 5.513128 3.936494 7.721233\n",
      "wall wall 2.134485 1.646756 2.766668\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.789031 1.372426 2.332098\n",
      "wnon wnon 5.928437 4.037413 8.705168\n",
      "wspl wspl 4.836457 2.986319 7.832824\n",
      "wtru wtru 5.513128 3.936494 7.721233\n",
      "wall wall 2.134485 1.646756 2.766668\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.792490 1.375198 2.336406\n",
      "wnon wnon 5.937404 4.043488 8.718406\n",
      "wspl wspl 4.843999 2.990994 7.844994\n",
      "wtru wtru 5.521537 3.942487 7.733031\n",
      "wall wall 2.137853 1.649452 2.770869\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.792490 1.375198 2.336406\n",
      "wnon wnon 5.937404 4.043488 8.718406\n",
      "wspl wspl 4.843999 2.990994 7.844994\n",
      "wtru wtru 5.521537 3.942487 7.733031\n",
      "wall wall 2.137853 1.649452 2.770869\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.4e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.798427 1.379874 2.343940\n",
      "wnon wnon 5.858561 3.991118 8.599781\n",
      "wspl wspl 4.852687 2.996329 7.859143\n",
      "wtru wtru 5.479099 3.912559 7.672864\n",
      "wall wall 2.140756 1.651779 2.774485\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.805952 1.385809 2.353473\n",
      "wnon wnon 5.877782 4.004027 8.628393\n",
      "wspl wspl 4.869372 3.006576 7.886309\n",
      "wtru wtru 5.497352 3.925431 7.698742\n",
      "wall wall 2.148240 1.657687 2.783959\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.805952 1.385809 2.353473\n",
      "wnon wnon 5.877782 4.004027 8.628393\n",
      "wspl wspl 4.869372 3.006576 7.886309\n",
      "wtru wtru 5.497352 3.925431 7.698742\n",
      "wall wall 2.148240 1.657687 2.783959\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.809077 1.388310 2.357368\n",
      "wnon wnon 5.885121 4.009008 8.639207\n",
      "wspl wspl 4.859972 3.001538 7.869076\n",
      "wtru wtru 5.497470 3.925776 7.698397\n",
      "wall wall 2.151000 1.659901 2.787396\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.809077 1.388310 2.357368\n",
      "wnon wnon 5.885121 4.009008 8.639207\n",
      "wspl wspl 4.859972 3.001538 7.869076\n",
      "wtru wtru 5.497470 3.925776 7.698397\n",
      "wall wall 2.151000 1.659901 2.787396\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.810798 1.389671 2.359546\n",
      "wnon wnon 6.013073 4.101318 8.815959\n",
      "wspl wspl 4.869320 3.007302 7.884237\n",
      "wtru wtru 5.579626 3.987250 7.807945\n",
      "wall wall 2.159553 1.666629 2.798265\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.810798 1.389671 2.359546\n",
      "wnon wnon 6.013073 4.101318 8.815959\n",
      "wspl wspl 4.869320 3.007302 7.884237\n",
      "wtru wtru 5.579626 3.987250 7.807945\n",
      "wall wall 2.159553 1.666629 2.798265\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.810798 1.389671 2.359546\n",
      "wnon wnon 6.013073 4.101318 8.815959\n",
      "wspl wspl 4.869320 3.007302 7.884237\n",
      "wtru wtru 5.579626 3.987250 7.807945\n",
      "wall wall 2.159553 1.666629 2.798265\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.810798 1.389671 2.359546\n",
      "wnon wnon 6.013073 4.101318 8.815959\n",
      "wspl wspl 4.869320 3.007302 7.884237\n",
      "wtru wtru 5.579626 3.987250 7.807945\n",
      "wall wall 2.159553 1.666629 2.798265\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.47e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.789311 1.375118 2.328261\n",
      "wnon wnon 5.917671 4.042236 8.663231\n",
      "wspl wspl 4.818687 2.978226 7.796504\n",
      "wtru wtru 5.503246 3.937807 7.691011\n",
      "wall wall 2.133818 1.649148 2.760929\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.47e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.789311 1.375118 2.328261\n",
      "wnon wnon 5.917671 4.042236 8.663231\n",
      "wspl wspl 4.818687 2.978226 7.796504\n",
      "wtru wtru 5.503246 3.937807 7.691011\n",
      "wall wall 2.133818 1.649148 2.760929\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.47e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, FAT1, CUL3, BAI3.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 4.71e+03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.789311 1.375118 2.328261\n",
      "wnon wnon 5.917671 4.042236 8.663231\n",
      "wspl wspl 4.818687 2.978226 7.796504\n",
      "wtru wtru 5.503246 3.937807 7.691011\n",
      "wall wall 2.133818 1.649148 2.760929\n",
      "[1] \"PD30988\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.51).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH2, NOTCH3, FAT1, NOTCH1.”    Regression model for indels (theta = 0.671)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.574279 1.139845  2.174291\n",
      "wnon wnon 4.368679 2.671360  7.144435\n",
      "wspl wspl 5.875354 3.299114 10.463348\n",
      "wtru wtru 4.887197 3.205168  7.451932\n",
      "wall wall 1.846969 1.346125  2.534158\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.13).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, ARID2, KMT2D, CCND1.”    Regression model for indels (theta = 1.03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.761251 1.386645  2.237058\n",
      "wnon wnon 7.214448 5.240048  9.932784\n",
      "wspl wspl 8.626102 5.751755 12.936858\n",
      "wtru wtru 7.640857 5.724853 10.198112\n",
      "wall wall 2.263242 1.792312  2.857909\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.81).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, ARID2, CCND1.”    Regression model for indels (theta = 0.839)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.893235 1.508367  2.376305\n",
      "wnon wnon 7.817309 5.773263 10.585056\n",
      "wspl wspl 9.667093 6.628345 14.098946\n",
      "wtru wtru 8.386192 6.382680 11.018603\n",
      "wall wall 2.445276 1.958600  3.052881\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.93).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, ARID2, CCND1.”    Regression model for indels (theta = 0.682)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.973239 1.579206  2.465589\n",
      "wnon wnon 8.143779 6.051851 10.958818\n",
      "wspl wspl 9.615551 6.650542 13.902450\n",
      "wtru wtru 8.606590 6.586590 11.246092\n",
      "wall wall 2.530879 2.035560  3.146725\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.92).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.682)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.034914 1.629973  2.540456\n",
      "wnon wnon  8.606010 6.414819 11.545675\n",
      "wspl wspl 10.046983 6.972919 14.476273\n",
      "wtru wtru  9.058064 6.945955 11.812417\n",
      "wall wall  2.617449 2.106609  3.252165\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.89).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.682)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.092491 1.677004  2.610917\n",
      "wnon wnon  9.072961 6.776581 12.147517\n",
      "wspl wspl 10.238936 7.120391 14.723322\n",
      "wtru wtru  9.441337 7.248905 12.296869\n",
      "wall wall  2.697506 2.172038  3.350097\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.9).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.762)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.114837 1.695400  2.638042\n",
      "wnon wnon  9.227775 6.894289 12.351067\n",
      "wspl wspl 10.525923 7.336829 15.101219\n",
      "wtru wtru  9.640094 7.405035 12.549759\n",
      "wall wall  2.727755 2.196881  3.386914\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.19).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.762)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.055179 1.652178  2.556481\n",
      "wnon wnon  8.928239 6.687514 11.919743\n",
      "wspl wspl 10.604128 7.422473 15.149605\n",
      "wtru wtru  9.456116 7.283769 12.276354\n",
      "wall wall  2.654690 2.144124  3.286833\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.762)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.047365 1.647641  2.544065\n",
      "wnon wnon  9.002358 6.752351 12.002109\n",
      "wspl wspl 10.562804 7.398374 15.080723\n",
      "wtru wtru  9.493650 7.320779 12.311447\n",
      "wall wall  2.647359 2.140431  3.274345\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.046753 1.647157  2.543288\n",
      "wnon wnon  9.041290 6.783477 12.050592\n",
      "wspl wspl 10.563607 7.398959 15.081822\n",
      "wtru wtru  9.520308 7.342397 12.344233\n",
      "wall wall  2.649338 2.142072  3.276731\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.046753 1.647157  2.543288\n",
      "wnon wnon  9.041290 6.783477 12.050592\n",
      "wspl wspl 10.563607 7.398959 15.081822\n",
      "wtru wtru  9.520308 7.342397 12.344233\n",
      "wall wall  2.649338 2.142072  3.276731\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.049956 1.649801  2.547168\n",
      "wnon wnon  9.003908 6.757250 11.997538\n",
      "wspl wspl 10.676302 7.486164 15.225878\n",
      "wtru wtru  9.528416 7.350183 12.352170\n",
      "wall wall  2.653069 2.145165  3.281228\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.053793 1.652962  2.551823\n",
      "wnon wnon  9.017490 6.767349 12.015802\n",
      "wspl wspl 10.690561 7.496355 15.245820\n",
      "wtru wtru  9.542373 7.360910 12.370329\n",
      "wall wall  2.656873 2.148300  3.285841\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.055261 1.654206  2.553551\n",
      "wnon wnon  9.063667 6.803889 12.073987\n",
      "wspl wspl 10.699977 7.503060 15.259041\n",
      "wtru wtru  9.576779 7.388503 12.413164\n",
      "wall wall  2.660873 2.151617  3.290662\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.057977 1.656449  2.556838\n",
      "wnon wnon  9.143673 6.866471 12.176089\n",
      "wspl wspl 10.718246 7.515895 15.285045\n",
      "wtru wtru  9.637414 7.436791 12.489224\n",
      "wall wall  2.668181 2.157620  3.299557\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.060556 1.658584  2.559949\n",
      "wnon wnon  9.152858 6.873328 12.188390\n",
      "wspl wspl 10.726268 7.521757 15.296004\n",
      "wtru wtru  9.646396 7.443739 12.500835\n",
      "wall wall  2.670692 2.159698  3.302589\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.060556 1.658584  2.559949\n",
      "wnon wnon  9.152858 6.873328 12.188390\n",
      "wspl wspl 10.726268 7.521757 15.296004\n",
      "wtru wtru  9.646396 7.443739 12.500835\n",
      "wall wall  2.670692 2.159698  3.302589\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.060556 1.658584  2.559949\n",
      "wnon wnon  9.152858 6.873328 12.188390\n",
      "wspl wspl 10.726268 7.521757 15.296004\n",
      "wtru wtru  9.646396 7.443739 12.500835\n",
      "wall wall  2.670692 2.159698  3.302589\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.060556 1.658584  2.559949\n",
      "wnon wnon  9.152858 6.873328 12.188390\n",
      "wspl wspl 10.726268 7.521757 15.296004\n",
      "wtru wtru  9.646396 7.443739 12.500835\n",
      "wall wall  2.670692 2.159698  3.302589\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.065217 1.662411  2.565623\n",
      "wnon wnon  9.169594 6.885753 12.210932\n",
      "wspl wspl 10.744568 7.534741 15.321791\n",
      "wtru wtru  9.663769 7.457053 12.523505\n",
      "wall wall  2.675357 2.163533  3.308263\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.067366 1.664191  2.568216\n",
      "wnon wnon  9.176187 6.890696 12.219726\n",
      "wspl wspl 10.722130 7.520557 15.286643\n",
      "wtru wtru  9.662395 7.456289 12.521227\n",
      "wall wall  2.677237 2.165092  3.310527\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.831)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.26).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071492 1.667601  2.573205\n",
      "wnon wnon  9.164365 6.882590 12.202614\n",
      "wspl wspl 10.732438 7.528133 15.300635\n",
      "wtru wtru  9.656632 7.452228 12.513111\n",
      "wall wall  2.680241 2.167574  3.314161\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.075660 1.671030  2.578268\n",
      "wnon wnon  9.179382 6.893752 12.222814\n",
      "wspl wspl 10.751232 7.541301 15.327459\n",
      "wtru wtru  9.672859 7.464650 12.534304\n",
      "wall wall  2.684388 2.170989  3.319197\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.075660 1.671030  2.578268\n",
      "wnon wnon  9.179382 6.893752 12.222814\n",
      "wspl wspl 10.751232 7.541301 15.327459\n",
      "wtru wtru  9.672859 7.464650 12.534304\n",
      "wall wall  2.684388 2.170989  3.319197\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.078914 1.673718  2.582206\n",
      "wnon wnon  9.191039 6.902439 12.238455\n",
      "wspl wspl 10.764011 7.550391 15.345421\n",
      "wtru wtru  9.684966 7.473966 12.550039\n",
      "wall wall  2.687595 2.173638  3.323078\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.078914 1.673718  2.582206\n",
      "wnon wnon  9.191039 6.902439 12.238455\n",
      "wspl wspl 10.764011 7.550391 15.345421\n",
      "wtru wtru  9.684966 7.473966 12.550039\n",
      "wall wall  2.687595 2.173638  3.323078\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.078914 1.673718  2.582206\n",
      "wnon wnon  9.191039 6.902439 12.238455\n",
      "wspl wspl 10.764011 7.550391 15.345421\n",
      "wtru wtru  9.684966 7.473966 12.550039\n",
      "wall wall  2.687595 2.173638  3.323078\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.078914 1.673718  2.582206\n",
      "wnon wnon  9.191039 6.902439 12.238455\n",
      "wspl wspl 10.764011 7.550391 15.345421\n",
      "wtru wtru  9.684966 7.473966 12.550039\n",
      "wall wall  2.687595 2.173638  3.323078\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, NOTCH3, FAT1, ARID1A, KMT2D, CCND1, ARID2.”    Regression model for indels (theta = 0.797)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085423 1.679079  2.590104\n",
      "wnon wnon  9.183208 6.897659 12.226075\n",
      "wspl wspl 10.755036 7.545873 15.329015\n",
      "wtru wtru  9.676703 7.468514 12.537778\n",
      "wall wall  2.693320 2.178370  3.330000\n",
      "[1] \"PD30273\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.45e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A.”    Regression model for indels (theta = 1.86)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.742088 1.212192 2.503622\n",
      "wnon wnon 4.292175 2.401783 7.670452\n",
      "wspl wspl 4.636045 2.202579 9.758065\n",
      "wtru wtru 4.406391 2.660446 7.298131\n",
      "wall wall 1.943787 1.358736 2.780751\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 6.32e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.59)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.871344 1.440161 2.431621\n",
      "wnon wnon 5.045342 3.404266 7.477523\n",
      "wspl wspl 5.646135 3.338738 9.548173\n",
      "wtru wtru 5.222870 3.675927 7.420814\n",
      "wall wall 2.118739 1.636274 2.743460\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.2e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.39)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.872586 1.462677 2.397371\n",
      "wnon wnon 5.513737 3.838321 7.920467\n",
      "wspl wspl 5.810461 3.537998 9.542533\n",
      "wtru wtru 5.599223 4.039327 7.761515\n",
      "wall wall 2.154136 1.688682 2.747884\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.33e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, TP53, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.39)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.985099 1.555301 2.533669\n",
      "wnon wnon 6.525331 4.611666 9.233093\n",
      "wspl wspl 6.039867 3.736116 9.764149\n",
      "wtru wtru 6.381190 4.655926 8.745755\n",
      "wall wall 2.323758 1.827375 2.954979\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.39e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.39)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.023072 1.590175  2.573818\n",
      "wnon wnon 6.719546 4.785427  9.435376\n",
      "wspl wspl 6.445479 4.048371 10.261956\n",
      "wtru wtru 6.638745 4.876223  9.038334\n",
      "wall wall 2.379535 1.877274  3.016176\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.3e+04).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.34)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.002088 1.580193  2.536623\n",
      "wnon wnon 6.820612 4.888516  9.516332\n",
      "wspl wspl 6.669701 4.239511 10.492933\n",
      "wtru wtru 6.775969 5.007968  9.168140\n",
      "wall wall 2.367342 1.875368  2.988378\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.86e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.034715 1.608613  2.573686\n",
      "wnon wnon 6.759259 4.857273  9.406015\n",
      "wspl wspl 6.881461 4.397489 10.768534\n",
      "wtru wtru 6.794873 5.033394  9.172798\n",
      "wall wall 2.397433 1.902134  3.021704\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.85e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.050870 1.621699  2.593618\n",
      "wnon wnon 6.795413 4.888608  9.445968\n",
      "wspl wspl 7.003042 4.494175 10.912479\n",
      "wtru wtru 6.856084 5.084045  9.245765\n",
      "wall wall 2.418969 1.919607  3.048234\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.87e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.069373 1.636784  2.616290\n",
      "wnon wnon 7.035087 5.070677  9.760522\n",
      "wspl wspl 6.850080 4.398919 10.667073\n",
      "wtru wtru 6.980053 5.183065  9.400064\n",
      "wall wall 2.447129 1.942476  3.082891\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.88e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.090593 1.653756  2.642821\n",
      "wnon wnon 7.016602 5.059206  9.731311\n",
      "wspl wspl 6.812171 4.377508 10.600935\n",
      "wtru wtru 6.955759 5.165958  9.365657\n",
      "wall wall 2.465713 1.957366  3.106083\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.82e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.112331 1.671231  2.669854\n",
      "wnon wnon 7.047499 5.081770  9.773610\n",
      "wspl wspl 6.866999 4.412887 10.685901\n",
      "wtru wtru 6.993908 5.194218  9.417155\n",
      "wall wall 2.486389 1.974001  3.131776\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.79e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.120853 1.678147  2.680346\n",
      "wnon wnon 7.177724 5.178458  9.948855\n",
      "wspl wspl 6.895585 4.431372 10.730108\n",
      "wtru wtru 7.093705 5.270036  9.548446\n",
      "wall wall 2.499598 1.984677  3.148113\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.76e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.128880 1.684578  2.690364\n",
      "wnon wnon 7.200153 5.194386  9.980429\n",
      "wspl wspl 6.917639 4.445514 10.764497\n",
      "wtru wtru 7.116009 5.286382  9.578874\n",
      "wall wall 2.507720 1.991196  3.158232\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.75e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.140110 1.693656  2.704251\n",
      "wnon wnon 7.231516 5.216779 10.024350\n",
      "wspl wspl 6.948089 4.465147 10.811723\n",
      "wtru wtru 7.147077 5.309293  9.621000\n",
      "wall wall 2.518945 2.000277  3.172102\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.75e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.142529 1.695635  2.707204\n",
      "wnon wnon 7.238262 5.221632 10.033727\n",
      "wspl wspl 6.954355 4.469228 10.821346\n",
      "wtru wtru 7.153671 5.314200  9.629862\n",
      "wall wall 2.521317 2.002217  3.175001\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.76e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.147599 1.699743  2.713458\n",
      "wnon wnon 7.252414 5.231752 10.053518\n",
      "wspl wspl 6.967876 4.477968 10.842259\n",
      "wtru wtru 7.167623 5.324508  9.648744\n",
      "wall wall 2.526367 2.006311  3.181227\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.73e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.161139 1.710631  2.730292\n",
      "wnon wnon 7.568766 5.473734 10.465658\n",
      "wspl wspl 7.018571 4.510751 10.920651\n",
      "wtru wtru 7.404942 5.509255  9.952920\n",
      "wall wall 2.558738 2.032396  3.221390\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.73e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.170435 1.718210  2.741684\n",
      "wnon wnon 7.595622 5.493057 10.502981\n",
      "wspl wspl 7.041702 4.525854 10.956070\n",
      "wtru wtru 7.430607 5.528347  9.987419\n",
      "wall wall 2.567905 2.039868  3.232628\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.178052 1.724324  2.751172\n",
      "wnon wnon 7.617699 5.508788 10.533956\n",
      "wspl wspl 7.062959 4.539476 10.989240\n",
      "wtru wtru 7.452422 5.544371 10.017116\n",
      "wall wall 2.575595 2.046050  3.242192\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.185559 1.730352  2.760519\n",
      "wnon wnon 7.639365 5.524233 10.564344\n",
      "wspl wspl 7.081081 4.551273 11.017076\n",
      "wtru wtru 7.472951 5.559500 10.044969\n",
      "wall wall 2.583183 2.052154  3.251625\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.185559 1.730352  2.760519\n",
      "wnon wnon 7.639365 5.524233 10.564344\n",
      "wspl wspl 7.081081 4.551273 11.017076\n",
      "wtru wtru 7.472951 5.559500 10.044969\n",
      "wall wall 2.583183 2.052154  3.251625\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.185559 1.730352  2.760519\n",
      "wnon wnon 7.639365 5.524233 10.564344\n",
      "wspl wspl 7.081081 4.551273 11.017076\n",
      "wtru wtru 7.472951 5.559500 10.044969\n",
      "wall wall 2.583183 2.052154  3.251625\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.185559 1.730352  2.760519\n",
      "wnon wnon 7.639365 5.524233 10.564344\n",
      "wspl wspl 7.081081 4.551273 11.017076\n",
      "wtru wtru 7.472951 5.559500 10.044969\n",
      "wall wall 2.583183 2.052154  3.251625\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.71e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.186929 1.731465  2.762204\n",
      "wnon wnon 7.723895 5.589149 10.673995\n",
      "wspl wspl 7.087562 4.555579 11.026816\n",
      "wtru wtru 7.534202 5.607431 10.123031\n",
      "wall wall 2.589572 2.057312  3.259536\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.189062 1.733168  2.764875\n",
      "wnon wnon 7.792373 5.642505 10.761369\n",
      "wspl wspl 7.097097 4.561712 11.041639\n",
      "wtru wtru 7.585360 5.647900 10.187448\n",
      "wall wall 2.597105 2.063372  3.268898\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.194524 1.737588  2.771622\n",
      "wnon wnon 7.808429 5.654018 10.783759\n",
      "wspl wspl 7.110644 4.570532 11.062445\n",
      "wtru wtru 7.600607 5.659195 10.208029\n",
      "wall wall 2.602556 2.067787  3.275627\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.72e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.196327 1.739066  2.773817\n",
      "wnon wnon 7.813723 5.657845 10.791080\n",
      "wspl wspl 7.114972 4.573374 11.069031\n",
      "wtru wtru 7.605589 5.662920 10.214693\n",
      "wall wall 2.604315 2.069227  3.277772\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 9.73e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, NOTCH2, ARID1A, NOTCH3, FAT1, PIK3CA.”    Regression model for indels (theta = 1.58)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.200397 1.742346  2.778867\n",
      "wnon wnon 7.745533 5.609391 10.695152\n",
      "wspl wspl 7.120119 4.576760 11.076851\n",
      "wtru wtru 7.560326 5.629577 10.153255\n",
      "wall wall 2.605862 2.070496  3.279657\n",
      "[1] \"PD31182\"\n",
      "[1] 1\n",
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.34).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, FAT1, NOTCH1.”    Regression model for indels (theta = 0.604)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.217239 0.8250061 1.795953\n",
      "wnon wnon 4.165312 2.2960444 7.556399\n",
      "wspl wspl 2.626451 1.0665535 6.467792\n",
      "wtru wtru 3.660766 2.1388938 6.265485\n",
      "wall wall 1.408834 0.9627279 2.061655\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.46).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH2, NOTCH3.”    Regression model for indels (theta = 1.99)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.576208 1.197813  2.074139\n",
      "wnon wnon 5.997590 4.055380  8.869968\n",
      "wspl wspl 6.002507 3.446718 10.453447\n",
      "wtru wtru 5.998953 4.203734  8.560825\n",
      "wall wall 1.921102 1.469289  2.511848\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.49).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.64)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.766705 1.356072  2.301682\n",
      "wnon wnon 7.346858 5.097917 10.587918\n",
      "wspl wspl 7.164575 4.331969 11.849377\n",
      "wtru wtru 7.294442 5.226396 10.180797\n",
      "wall wall 2.182630 1.684905  2.827384\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.63).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.4)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.896197 1.465747  2.453058\n",
      "wnon wnon 7.425094 5.210790 10.580358\n",
      "wspl wspl 8.238127 5.135656 13.214812\n",
      "wtru wtru 7.656106 5.546438 10.568216\n",
      "wall wall 2.333589 1.813573  3.002713\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.63).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.4)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.972088 1.525938  2.548682\n",
      "wnon wnon 8.052779 5.679027 11.418725\n",
      "wspl wspl 9.216080 5.842403 14.537877\n",
      "wtru wtru 8.390481 6.108872 11.524250\n",
      "wall wall 2.458566 1.912711  3.160199\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.01)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 2.000993 1.551643  2.580475\n",
      "wnon wnon 8.141198 5.763146 11.500508\n",
      "wspl wspl 9.877619 6.325712 15.423934\n",
      "wtru wtru 8.640771 6.313726 11.825494\n",
      "wall wall 2.496816 1.946390  3.202899\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.02)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  1.996195 1.550653  2.569754\n",
      "wnon wnon  8.414160 5.979599 11.839939\n",
      "wspl wspl 10.092310 6.489398 15.695555\n",
      "wtru wtru  8.896225 6.520239 12.138025\n",
      "wall wall  2.504120 1.955519  3.206626\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.02)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  1.974510 1.538512  2.534066\n",
      "wnon wnon  8.355054 5.958845 11.714841\n",
      "wspl wspl 10.240361 6.616513 15.848981\n",
      "wtru wtru  8.892010 6.538710 12.092270\n",
      "wall wall  2.479011 1.941812  3.164825\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.65).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.02)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  1.983896 1.548670  2.541434\n",
      "wnon wnon  8.404844 6.009123 11.755693\n",
      "wspl wspl 10.100687 6.539832 15.600384\n",
      "wtru wtru  8.889818 6.550864 12.063883\n",
      "wall wall  2.487431 1.951760  3.170122\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.67).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.02)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.939175 1.517708  2.477684\n",
      "wnon wnon 8.253940 5.918814 11.510334\n",
      "wspl wspl 9.890909 6.413772 15.253128\n",
      "wtru wtru 8.718654 6.441553 11.800713\n",
      "wall wall 2.431653 1.912962  3.090984\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.02)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  1.948580 1.525279  2.489356\n",
      "wnon wnon  8.340823 5.990725 11.612841\n",
      "wspl wspl 10.421609 6.809166 15.950549\n",
      "wtru wtru  8.933310 6.612262 12.069097\n",
      "wall wall  2.457967 1.934083  3.123754\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.67).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 2.02)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  1.983675 1.553241  2.533391\n",
      "wnon wnon  8.289615 5.957614 11.534437\n",
      "wspl wspl 10.809824 7.081042 16.502132\n",
      "wtru wtru  8.996672 6.662103 12.149334\n",
      "wall wall  2.494341 1.963165  3.169239\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.67).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  1.990855 1.559036  2.542279\n",
      "wnon wnon  8.422945 6.057209 11.712656\n",
      "wspl wspl 11.159491 7.342009 16.961874\n",
      "wtru wtru  9.197663 6.817393 12.408997\n",
      "wall wall  2.514052 1.979007  3.193753\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.008114 1.572806  2.563904\n",
      "wnon wnon  8.542588 6.145461 11.874750\n",
      "wspl wspl 11.444340 7.548164 17.351626\n",
      "wtru wtru  9.367951 6.947490 12.631686\n",
      "wall wall  2.540659 2.000274  3.227031\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.014462 1.577895  2.571818\n",
      "wnon wnon  8.564669 6.161177 11.905771\n",
      "wspl wspl 11.472995 7.567130 17.394921\n",
      "wtru wtru  9.392086 6.965243 12.664494\n",
      "wall wall  2.547014 2.005376  3.234946\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.022673 1.584453  2.582093\n",
      "wnon wnon  8.718806 6.279808 12.105082\n",
      "wspl wspl 11.521725 7.599512 17.468248\n",
      "wtru wtru  9.514441 7.061021 12.820326\n",
      "wall wall  2.567368 2.021659  3.260382\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.018631 1.581258  2.576979\n",
      "wnon wnon  8.815827 6.352530 12.234307\n",
      "wspl wspl 11.511803 7.593141 17.452805\n",
      "wtru wtru  9.582796 7.113488 12.909276\n",
      "wall wall  2.567368 2.021659  3.260382\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.022386 1.584289  2.581627\n",
      "wnon wnon  8.803839 6.345007 12.215524\n",
      "wspl wspl 11.516753 7.597044 17.458842\n",
      "wtru wtru  9.574125 7.107808 12.896223\n",
      "wall wall  2.570491 2.024190  3.264230\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.050548 1.606811  2.616827\n",
      "wnon wnon  8.967092 6.465744 12.436113\n",
      "wspl wspl 11.648086 7.684075 17.657025\n",
      "wtru wtru  9.728211 7.224045 13.100429\n",
      "wall wall  2.604912 2.051755  3.307201\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.050548 1.606811  2.616827\n",
      "wnon wnon  8.967092 6.465744 12.436113\n",
      "wspl wspl 11.648086 7.684075 17.657025\n",
      "wtru wtru  9.728211 7.224045 13.100429\n",
      "wall wall  2.604912 2.051755  3.307201\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.057680 1.612465  2.625821\n",
      "wnon wnon  9.107270 6.569131 12.626078\n",
      "wspl wspl 11.559344 7.628898 17.514777\n",
      "wtru wtru  9.809661 7.285732 13.207933\n",
      "wall wall  2.615126 2.059860  3.320074\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.057680 1.612465  2.625821\n",
      "wnon wnon  9.107270 6.569131 12.626078\n",
      "wspl wspl 11.559344 7.628898 17.514777\n",
      "wtru wtru  9.809661 7.285732 13.207933\n",
      "wall wall  2.615126 2.059860  3.320074\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.064292 1.617770  2.634058\n",
      "wnon wnon  9.186115 6.629503 12.728661\n",
      "wspl wspl 11.591591 7.650503 17.562896\n",
      "wtru wtru  9.874529 7.336197 13.291126\n",
      "wall wall  2.627527 2.069811  3.335522\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.071566 1.623573  2.643173\n",
      "wnon wnon  9.209320 6.646100 12.761105\n",
      "wspl wspl 11.535596 7.617781 17.468336\n",
      "wtru wtru  9.879481 7.340789 13.296140\n",
      "wall wall  2.635284 2.076012  3.345222\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.086460 1.635522  2.661727\n",
      "wnon wnon  9.335889 6.742087 12.927574\n",
      "wspl wspl 11.450412 7.567705 17.325191\n",
      "wtru wtru  9.950242 7.397295 13.384261\n",
      "wall wall  2.655796 2.092522  3.370695\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.085803 1.635020  2.660869\n",
      "wnon wnon  9.386964 6.781772 12.992931\n",
      "wspl wspl 11.453336 7.569639 17.329614\n",
      "wtru wtru  9.986887 7.426221 13.430508\n",
      "wall wall  2.658286 2.094544  3.373758\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.090319 1.638635  2.666508\n",
      "wnon wnon  9.301045 6.721082 12.871354\n",
      "wspl wspl 11.460335 7.574444 17.339792\n",
      "wtru wtru  9.924061 7.380123 13.344897\n",
      "wall wall  2.659949 2.095898  3.375799\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.090319 1.638635  2.666508\n",
      "wnon wnon  9.301045 6.721082 12.871354\n",
      "wspl wspl 11.460335 7.574444 17.339792\n",
      "wtru wtru  9.924061 7.380123 13.344897\n",
      "wall wall  2.659949 2.095898  3.375799\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.091342 1.639469  2.667761\n",
      "wnon wnon  9.314102 6.730371 12.889705\n",
      "wspl wspl 11.711349 7.758919 17.677168\n",
      "wtru wtru 10.007609 7.444424 13.453322\n",
      "wall wall  2.665529 2.100389  3.382730\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.091342 1.639469  2.667761\n",
      "wnon wnon  9.314102 6.730371 12.889705\n",
      "wspl wspl 11.711349 7.758919 17.677168\n",
      "wtru wtru 10.007609 7.444424 13.453322\n",
      "wall wall  2.665529 2.100389  3.382730\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.091342 1.639469  2.667761\n",
      "wnon wnon  9.314102 6.730371 12.889705\n",
      "wspl wspl 11.711349 7.758919 17.677168\n",
      "wtru wtru 10.007609 7.444424 13.453322\n",
      "wall wall  2.665529 2.100389  3.382730\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.091342 1.639469  2.667761\n",
      "wnon wnon  9.314102 6.730371 12.889705\n",
      "wspl wspl 11.711349 7.758919 17.677168\n",
      "wtru wtru 10.007609 7.444424 13.453322\n",
      "wall wall  2.665529 2.100389  3.382730\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.105866 1.651066  2.685945\n",
      "wnon wnon  9.367251 6.768266 12.964233\n",
      "wspl wspl 11.774664 7.801008 17.772410\n",
      "wtru wtru 10.064123 7.486016 13.530105\n",
      "wall wall  2.680197 2.112121  3.401063\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.105866 1.651066  2.685945\n",
      "wnon wnon  9.367251 6.768266 12.964233\n",
      "wspl wspl 11.774664 7.801008 17.772410\n",
      "wtru wtru 10.064123 7.486016 13.530105\n",
      "wall wall  2.680197 2.112121  3.401063\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.105866 1.651066  2.685945\n",
      "wnon wnon  9.367251 6.768266 12.964233\n",
      "wspl wspl 11.774664 7.801008 17.772410\n",
      "wtru wtru 10.064123 7.486016 13.530105\n",
      "wall wall  2.680197 2.112121  3.401063\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.105866 1.651066  2.685945\n",
      "wnon wnon  9.367251 6.768266 12.964233\n",
      "wspl wspl 11.774664 7.801008 17.772410\n",
      "wtru wtru 10.064123 7.486016 13.530105\n",
      "wall wall  2.680197 2.112121  3.401063\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.105866 1.651066  2.685945\n",
      "wnon wnon  9.367251 6.768266 12.964233\n",
      "wspl wspl 11.774664 7.801008 17.772410\n",
      "wtru wtru 10.064123 7.486016 13.530105\n",
      "wall wall  2.680197 2.112121  3.401063\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, ARID1A, NOTCH3, NOTCH2.”    Regression model for indels (theta = 1.92)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.105866 1.651066  2.685945\n",
      "wnon wnon  9.367251 6.768266 12.964233\n",
      "wspl wspl 11.774664 7.801008 17.772410\n",
      "wtru wtru 10.064123 7.486016 13.530105\n",
      "wall wall  2.680197 2.112121  3.401063\n"
     ]
    }
   ],
   "source": [
    "dfdnds.patient <- data.frame()\n",
    "dfdnds.genes.patient <- data.frame()\n",
    "i <- 1\n",
    "for (p in unique(df$donor)){\n",
    "  print(p)\n",
    "  print(i)\n",
    "  for (cutoff in areacutoff){\n",
    "    print(cutoff)\n",
    "    x1 <- df %>%\n",
    "      filter(donor == p) %>%\n",
    "      filter(sumvaf < cutoff) #filter for mutations with sumvaf < cutoff\n",
    "    x <- dndscv(x1, gene_list = target_genes,\n",
    "                outp = 3, max_muts_per_gene_per_sample = Inf, \n",
    "                max_coding_muts_per_sample = Inf)\n",
    "    out <- x$globaldnds %>%\n",
    "        mutate(areacutoff = cutoff, nmutations = length(x1$donor), patient = p)\n",
    "    dfdnds.patient <- rbind(dfdnds.patient, out)\n",
    "    combined <- left_join(x$sel_cv, x$sel_loc, by = c(\"gene_name\", \"n_syn\", \"n_mis\", \"n_non\", \"n_spl\"))\n",
    "    out2 <- combined %>%\n",
    "      mutate(areacutoff = cutoff, patient = p)\n",
    "    dfdnds.genes.patient <- rbind(dfdnds.genes.patient, out2)\n",
    "    print(x$globaldnds)\n",
    "  }\n",
    "  i + i +1\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "write_csv(dfdnds.patient, \"data/esophagus/dnds_patient_combined.csv\")\n",
    "write_csv(dfdnds.genes.patient, \"data/esophagus/dnds_patient_genes_combined.csv\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The genes under positive selection naturally have many more mutations, so we are biased to detect positively selected genes and measure their selective coefficient. To demonstarte that our theory is correct for neutral genes, we'll remove all the highly selected genes and calculate the global dN/dS on the remaining genes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [],
   "source": [
    "target_genesneutral <- target_genes[!target_genes %in% c(\"NOTCH1\", \"NOTCH2\", \"NOTCH3\", \"TP53\",\"NFE2L2\", \"ARID1A\", \"CDH1\",\n",
    "                       \"CREBBP\", \"FAT1\", \"SALL1\", \"KMT2D\", \"ARID2\",\"CCND2\", \"CCND1\", \"KMT2A\", \"SCN1A\", \"FBXW7\",\n",
    "                      \"KMT2C\", \"PIK3CA\", \"SPHKAP\", \"CUL3\", \"TP63\", \"AJUBA\", \"EPHA2\", \"IRF6\", \"FGFR2\")]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 0.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 33.1).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.84)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0100176 0.8408747 1.213184\n",
      "wnon wnon 1.0684401 0.7187210 1.588327\n",
      "wspl wspl 0.7572592 0.3891782 1.473468\n",
      "wtru wtru 0.9736593 0.6855562 1.382837\n",
      "wall wall 1.0083093 0.8415208 1.208155\n",
      "[1] 0.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 8.12).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 9.03)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 0.9999455 0.8701231 1.149137\n",
      "wnon wnon 0.9632208 0.7077574 1.310893\n",
      "wspl wspl 0.9249877 0.5666818 1.509846\n",
      "wtru wtru 0.9527509 0.7274363 1.247854\n",
      "wall wall 0.9958605 0.8682289 1.142254\n",
      "[1] 0.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.39).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.47)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0247478 0.8959636 1.172043\n",
      "wnon wnon 0.9575934 0.7084771 1.294305\n",
      "wspl wspl 1.0383211 0.6563554 1.642572\n",
      "wtru wtru 0.9794889 0.7545271 1.271523\n",
      "wall wall 1.0200816 0.8935223 1.164567\n",
      "[1] 0.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.91).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.23)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0358784 0.9077000 1.182157\n",
      "wnon wnon 0.9877241 0.7361057 1.325352\n",
      "wspl wspl 1.0421043 0.6649171 1.633259\n",
      "wtru wtru 1.0026286 0.7769416 1.293873\n",
      "wall wall 1.0324630 0.9063580 1.176114\n",
      "[1] 0.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.79).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.23)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0384000 0.9109990 1.183618\n",
      "wnon wnon 0.9865943 0.7372096 1.320341\n",
      "wspl wspl 1.0401542 0.6639272 1.629577\n",
      "wtru wtru 1.0010983 0.7771481 1.289584\n",
      "wall wall 1.0346182 0.9093239 1.177177\n",
      "[1] 0.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.67).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.72)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0424065 0.9147763 1.187844\n",
      "wnon wnon 0.9874153 0.7378828 1.321333\n",
      "wspl wspl 1.0309698 0.6580909 1.615125\n",
      "wtru wtru 0.9992797 0.7757862 1.287159\n",
      "wall wall 1.0381678 0.9126856 1.180902\n",
      "[1] 0.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.33).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.72)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0399755 0.9129232 1.184710\n",
      "wnon wnon 0.9858424 0.7381151 1.316712\n",
      "wspl wspl 1.0220834 0.6525530 1.600873\n",
      "wtru wtru 0.9956614 0.7739165 1.280941\n",
      "wall wall 1.0356533 0.9107571 1.177677\n",
      "[1] 0.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.86).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.72)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0260004 0.9012483 1.168021\n",
      "wnon wnon 0.9689247 0.7256940 1.293679\n",
      "wspl wspl 1.0106427 0.6453622 1.582675\n",
      "wtru wtru 0.9801780 0.7621348 1.260602\n",
      "wall wall 1.0215119 0.8989081 1.160838\n",
      "[1] 0.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.79).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.72)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0203257 0.8965364 1.161207\n",
      "wnon wnon 0.9809767 0.7361381 1.307248\n",
      "wspl wspl 0.9943608 0.6350268 1.557026\n",
      "wtru wtru 0.9846149 0.7665185 1.264766\n",
      "wall wall 1.0169664 0.8951830 1.155318\n",
      "[1] 0.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.95).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 8.55)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0171344 0.8938455 1.157429\n",
      "wnon wnon 0.9789186 0.7346292 1.304443\n",
      "wspl wspl 0.9918764 0.6334611 1.553085\n",
      "wtru wtru 0.9824419 0.7648698 1.261904\n",
      "wall wall 1.0138748 0.8925748 1.151659\n",
      "[1] 0.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.94).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 8.55)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0185207 0.8950940 1.158967\n",
      "wnon wnon 0.9778980 0.7338866 1.303041\n",
      "wspl wspl 0.9927799 0.6340427 1.554488\n",
      "wtru wtru 0.9819387 0.7644893 1.261239\n",
      "wall wall 1.0150777 0.8936572 1.152996\n",
      "[1] 0.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.94).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 8.55)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0185207 0.8950940 1.158967\n",
      "wnon wnon 0.9778980 0.7338866 1.303041\n",
      "wspl wspl 0.9927799 0.6340427 1.554488\n",
      "wtru wtru 0.9819387 0.7644893 1.261239\n",
      "wall wall 1.0150777 0.8936572 1.152996\n",
      "[1] 0.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.94).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0194774 0.8959574 1.160026\n",
      "wnon wnon 0.9769029 0.7331711 1.301660\n",
      "wspl wspl 0.9933747 0.6344270 1.555409\n",
      "wtru wtru 0.9813696 0.7640658 1.260476\n",
      "wall wall 1.0158832 0.8943861 1.153885\n",
      "[1] 0.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0210616 0.8973763 1.161795\n",
      "wnon wnon 0.9779215 0.7339318 1.303024\n",
      "wspl wspl 0.9945590 0.6351850 1.557259\n",
      "wtru wtru 0.9824329 0.7648906 1.261846\n",
      "wall wall 1.0174217 0.8957637 1.155603\n",
      "[1] 0.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0210616 0.8973763 1.161795\n",
      "wnon wnon 0.9779215 0.7339318 1.303024\n",
      "wspl wspl 0.9945590 0.6351850 1.557259\n",
      "wtru wtru 0.9824329 0.7648906 1.261846\n",
      "wall wall 1.0174217 0.8957637 1.155603\n",
      "[1] 0.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0218672 0.8980866 1.162708\n",
      "wnon wnon 0.9938677 0.7472287 1.321915\n",
      "wspl wspl 0.9954878 0.6357792 1.558711\n",
      "wtru wtru 0.9943068 0.7749852 1.275697\n",
      "wall wall 1.0193349 0.8974675 1.157751\n",
      "[1] 0.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0218672 0.8980866 1.162708\n",
      "wnon wnon 0.9938677 0.7472287 1.321915\n",
      "wspl wspl 0.9954878 0.6357792 1.558711\n",
      "wtru wtru 0.9943068 0.7749852 1.275697\n",
      "wall wall 1.0193349 0.8974675 1.157751\n",
      "[1] 0.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0218672 0.8980866 1.162708\n",
      "wnon wnon 0.9938677 0.7472287 1.321915\n",
      "wspl wspl 0.9954878 0.6357792 1.558711\n",
      "wtru wtru 0.9943068 0.7749852 1.275697\n",
      "wall wall 1.0193349 0.8974675 1.157751\n",
      "[1] 0.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0218672 0.8980866 1.162708\n",
      "wnon wnon 0.9938677 0.7472287 1.321915\n",
      "wspl wspl 0.9954878 0.6357792 1.558711\n",
      "wtru wtru 0.9943068 0.7749852 1.275697\n",
      "wall wall 1.0193349 0.8974675 1.157751\n",
      "[1] 0.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0232011 0.8992855 1.164191\n",
      "wnon wnon 0.9947379 0.7478804 1.323077\n",
      "wspl wspl 0.9964757 0.6364124 1.560252\n",
      "wtru wtru 0.9952088 0.7756867 1.276856\n",
      "wall wall 1.0206304 0.8986317 1.159192\n",
      "[1] 1.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.52\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.57\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.62\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.67\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.72\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.77\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.82\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.87\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.92\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 1.97\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 2.02\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 2.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 2.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 2.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.93).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0252228 0.9010850 1.166462\n",
      "wnon wnon 0.9960563 0.7488648 1.324843\n",
      "wspl wspl 0.9979467 0.6373540 1.562550\n",
      "wtru wtru 0.9965686 0.7767406 1.278611\n",
      "wall wall 1.0225928 0.9003782 1.161397\n",
      "[1] 2.22\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.92).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0264615 0.9022000 1.167838\n",
      "wnon wnon 0.9968641 0.7494700 1.325921\n",
      "wspl wspl 0.9988522 0.6379350 1.563962\n",
      "wtru wtru 0.9974028 0.7773898 1.279683\n",
      "wall wall 1.0237959 0.9014608 1.162733\n",
      "[1] 2.27\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.92).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0264615 0.9022000 1.167838\n",
      "wnon wnon 0.9968641 0.7494700 1.325921\n",
      "wspl wspl 0.9988522 0.6379350 1.563962\n",
      "wtru wtru 0.9974028 0.7773898 1.279683\n",
      "wall wall 1.0237959 0.9014608 1.162733\n",
      "[1] 2.32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 6.92).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0264615 0.9022000 1.167838\n",
      "wnon wnon 0.9968641 0.7494700 1.325921\n",
      "wspl wspl 0.9988522 0.6379350 1.563962\n",
      "wtru wtru 0.9974028 0.7773898 1.279683\n",
      "wall wall 1.0237959 0.9014608 1.162733\n",
      "[1] 2.37\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.02).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0241864 0.9003068 1.165111\n",
      "wnon wnon 0.9938637 0.7472880 1.321800\n",
      "wspl wspl 0.9969551 0.6367450 1.560938\n",
      "wtru wtru 0.9947005 0.7753510 1.276105\n",
      "wall wall 1.0214724 0.8995236 1.159954\n",
      "[1] 2.42\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.02).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0241864 0.9003068 1.165111\n",
      "wnon wnon 0.9938637 0.7472880 1.321800\n",
      "wspl wspl 0.9969551 0.6367450 1.560938\n",
      "wtru wtru 0.9947005 0.7753510 1.276105\n",
      "wall wall 1.0214724 0.8995236 1.159954\n",
      "[1] 2.47\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Mutations observed in contiguous sites within a sample. Please annotate or remove dinucleotide or complex substitutions for best results.”Warning message in dndscv(x1, gene_list = target_genesneutral, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.02).\n",
      "    No gene was excluded from the background indel model.\n",
      "    Regression model for indels (theta = 7.95)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 1.0241864 0.9003068 1.165111\n",
      "wnon wnon 0.9938637 0.7472880 1.321800\n",
      "wspl wspl 0.9969551 0.6367450 1.560938\n",
      "wtru wtru 0.9947005 0.7753510 1.276105\n",
      "wall wall 1.0214724 0.8995236 1.159954\n"
     ]
    }
   ],
   "source": [
    "minarea <- 0.02\n",
    "maxarea <- 2.5\n",
    "step <- 0.05\n",
    "areacutoff <- seq(minarea, maxarea, step)\n",
    "target_genes <- unique(df$gene)\n",
    "\n",
    "dfdnds <- data.frame()\n",
    "dfdnds.genes <- data.frame()\n",
    "\n",
    "for (cutoff in areacutoff){\n",
    "  print(cutoff)\n",
    "  x1 <- df %>%\n",
    "    filter(sumvaf < cutoff) #filter for mutations with ccf < cutoff\n",
    "  x <- dndscv(x1, gene_list = target_genesneutral,\n",
    "              outp = 3, max_muts_per_gene_per_sample = Inf, \n",
    "              max_coding_muts_per_sample = Inf)\n",
    "  out <- x$globaldnds %>%\n",
    "    mutate(areacutoff = cutoff, nmutations = length(x1$donor))\n",
    "  dfdnds <- rbind(dfdnds, out)\n",
    "  combined <- left_join(x$sel_cv, x$sel_loc, by = c(\"gene_name\", \"n_syn\", \"n_mis\", \"n_non\", \"n_spl\"))\n",
    "  out2 <- combined %>%\n",
    "    mutate(areacutoff = cutoff)\n",
    "  dfdnds.genes <- rbind(dfdnds.genes, out2)\n",
    "  print(x$globaldnds)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [],
   "source": [
    "write_csv(dfdnds, \"data/esophagus/dnds_combined_neutral.csv\")\n",
    "write_csv(dfdnds.genes, \"data/esophagus/dnds_genes_combined_neutral.csv\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now I'll calculate dN/dS in frequency bins in patient PD30988."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       ".\n",
       "PD30272 PD30273 PD30274 PD30986 PD30987 PD30988 PD31182 PD36712 PD36806 \n",
       "    400    1070     558    1018     787    1192     955     413     265 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "df %>%\n",
    "filter(sumvaf < 0.5) %>%\n",
    "pull(donor) %>% \n",
    "table(.)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 1.6e+03).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: .”    Regression model for indels (theta = 0.339)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow    cihigh\n",
      "wmis wmis 0.9718066 0.4595211  2.055201\n",
      "wnon wnon 3.9682702 1.2401686 12.697603\n",
      "wspl wspl 1.5148022 0.2861712  8.018368\n",
      "wtru wtru 2.8772008 1.0358244  7.991976\n",
      "wall wall 1.1409765 0.5479730  2.375715\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 0.85).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: FAT1, TP53, NOTCH1, NOTCH2.”    Regression model for indels (theta = 0.469)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.319462 0.9135547  1.905720\n",
      "wnon wnon 5.429014 3.1571575  9.335674\n",
      "wspl wspl 6.212536 2.8962645 13.325993\n",
      "wtru wtru 5.637857 3.4548808  9.200154\n",
      "wall wall 1.619303 1.1323780  2.315607\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.01).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, ARID1A.”    Regression model for indels (theta = 0.359)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle    cilow    cihigh\n",
      "wmis wmis  2.470444 1.484279  4.111823\n",
      "wnon wnon  8.205555 4.212112 15.985125\n",
      "wspl wspl 10.678278 4.178170 27.290805\n",
      "wtru wtru  8.768255 4.717201 16.298286\n",
      "wall wall  3.013721 1.827797  4.969103\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.57).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1, NOTCH3.”    Regression model for indels (theta = 0.068)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow    cihigh\n",
      "wmis wmis  6.802083  2.965639  15.60147\n",
      "wnon wnon 33.722910 13.273850  85.67482\n",
      "wspl wspl 50.282003 18.093600 139.73338\n",
      "wtru wtru 38.777797 15.936114  94.35911\n",
      "wall wall  9.063021  3.979592  20.63989\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 818).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: TP53, NOTCH1, FAT1.”    Regression model for indels (theta = 0.231)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow    cihigh\n",
      "wmis wmis  4.903392  2.250976  10.68126\n",
      "wnon wnon 29.475418 11.657255  74.52872\n",
      "wspl wspl 54.985386 19.768500 152.93991\n",
      "wtru wtru 37.181420 15.639135  88.39734\n",
      "wall wall  6.626079  3.068970  14.30607\n"
     ]
    }
   ],
   "source": [
    "df1 <- df %>%\n",
    "   #filter(sumvaf < 0.5) %>%\n",
    "   filter(donor == \"PD31182\") %>%\n",
    "   mutate(x = cut_number(sumvaf, 4)) %>%\n",
    "   group_by(x) %>%\n",
    "   mutate(medianvaf = median(sumvaf)) %>%\n",
    "   ungroup()\n",
    "\n",
    "df1 <- df %>%\n",
    "   #filter(sumvaf < 0.5) %>%\n",
    "   filter(donor == \"PD31182\") %>%\n",
    "   mutate(sumvaf = sumvaf * 2) %>% #times by 2 to convert to area\n",
    "   mutate(x = cut(sumvaf, breaks = c(0.004 * 2, 0.01 * 2, 0.03 * 2, 0.07 * 2, 0.2 * 2, 6 * 2))) %>%\n",
    "   group_by(x) %>%\n",
    "   mutate(medianvaf = median(sumvaf)) %>%\n",
    "   ungroup()\n",
    "\n",
    "target_genes <- unique(df1$gene)\n",
    "\n",
    "dfdnds <- data.frame()\n",
    "dfdnds.genes <- data.frame()\n",
    "bins <- sort(unique(df1$x))\n",
    "j <- 1\n",
    "\n",
    "for (f in sort(unique(df1$medianvaf))){\n",
    "  x1 <- df1 %>%\n",
    "    filter(medianvaf ==f) #filter for mutations with ccf < cutoff\n",
    "  x <- dndscv(x1, gene_list = target_genes,\n",
    "              outp = 3, max_muts_per_gene_per_sample = Inf, \n",
    "              max_coding_muts_per_sample = Inf)\n",
    "  out <- x$globaldnds %>%\n",
    "    mutate(medianbin = f, vafbin = bins[j], nmutations = length(x1$donor))\n",
    "  dfdnds <- rbind(dfdnds, out)\n",
    "  combined <- left_join(x$sel_cv, x$sel_loc, by = c(\"gene_name\", \"n_syn\", \"n_mis\", \"n_non\", \"n_spl\"))\n",
    "  out2 <- combined %>%\n",
    "    mutate(medianbin = f, vafbin = bins[j])\n",
    "  dfdnds.genes <- rbind(dfdnds.genes, out2)\n",
    "  print(x$globaldnds)\n",
    "  j <- j + 1\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [
    {
     "data": {},
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA0gAAANICAIAAAByhViMAAAABmJLR0QA/wD/AP+gvaeTAAAg\nAElEQVR4nO3deZyVdb3A8d8ZBhgWUUBxAyTRQcEFE7fMyO0ltwJEFFNESb2WVphZQOaCRbgV\nQpQpLmgkV0XECJfEQMm8ihBKSiyJECIosQixDMuc+8e8mjsNi0KHOTPfeb//mvM7z3nO98zr\n8PjxOctkstlsAgCg5ivI9wAAAOSGsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAg\nCGEHABBEYb4HqFKvvvrqhAkTioqKBg0alO9ZAAByrLqfsZswYUImk3nppZe2vWrixIlf/OIX\nmzdv3rBhw+OOO+6+++77xL3NmDHjjjvuGDZsWO4HBQDIt+p+xm7UqFHbXR82bNh1111XfvHN\nN9/8xje+sXnz5m9961tVNRoAQPVSTc/YrVq1asqUKX379n366ae3vXb+/Pnf+973MpnMT37y\nk3/84x8fffTRT3/604KCgkGDBm3YsKHqpwUAqA6q4xm7jRs3NmvWbCcbDB06dOvWrQMHDrzh\nhhvKVq6//voZM2b8z//8z9tvv33CCSdUyZgAANVLdTxjV1RUlP2Xnj17brvBCy+8UFhYeP31\n11dcHDNmTDabVXUAQK1VHc/Y7dzKlSsXLFhw3HHHbd269ZJLLnnuuefWr19/1FFHXXPNNX37\n9s1kMhU3/uc//3nssceWX/z444+rfF4AgCpS88Ju+fLlKaXmzZt/7nOfW7BgQdni9OnTL7/8\n8j/+8Y8PPfRQxY1LS0vLtwEAiK3mhd3q1atTSi+++OKxxx77hz/84aSTTtq8efNTTz119dVX\njxo1qm/fvl/4whfKN65fv/6AAQPKL86YMePFF1/Mw9AAAHtezQu7+vXrp5Qymcyzzz570EEH\nlS1efvnls2fP/tnPfvbiiy9WCrvbb7+9/OKIESOEHQAQVXX88MTOHXjggSmlFi1alFddmU6d\nOiXvogMAarGaF3b7779/69atP/zww7lz51Zcf+2111JKhx9+eJ7mAgDIs5oXdimlK664IqXU\nrVu3F154Yc2aNcuWLRs+fPgvf/nLRo0a9erVK9/TAQDkR817j11KacCAAX/4wx+mTp16zjnn\nlC8WFBTcf//9LVq0yONgAAB5VCPP2NWvX3/SpEm33357+/bt69Wrt88++3Tp0mXKlCkXXXRR\nvkcDAMibTDabzfcMVWfEiBH9+vXbe++9y74zBQAgkhp5xg4AgG0JOwCAIIQdAEAQwg4AIAhh\nBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHs\nAACCEHYAAEEIOwCAIIQdAEAQhfkeAABqkvHjx8+bN6+4uLhHjx75ngUqc8YOAHbB6NGjBw4c\nOHr06HwPAtsh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2\nAABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIO\nACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgB\nAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsA\ngCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcA\nEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAA\nghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBA\nEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAI\nQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABB\nCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAI\nYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh\n7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCE\nHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISw\nAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACCI6h52EyZMyGQyL730UqX1jz/++Pvf\n/35xcXGDBg322muvk0466YEHHsjHgAAA1UVhvgf4BKNGjdp28eOPPz7ppJPmzp1bvjJt2rRp\n06bNnj176NChVTgdAEA1Uk3P2K1atWrKlCl9+/Z9+umnt7126NChc+fOPeaYY1555ZUNGzYs\nWbLk1ltvTSkNGzbsrbfeqvJhAQCqhep4xm7jxo3NmjXbyQbjxo1LKY0dO7a4uDildNBBB918\n882zZ89+/PHHn3322WOPPbaKBgUAqE6q4xm7oqKi7L/07Nlz2w3ee++9gw8+uKzqyrVv3z6l\ntHHjxiqaEgCgmqmOZ+w+0bp16yqtbNq0qexF289//vMV1zdu3HjttdeWX3z77berYDwAgLyo\nkWFXyaJFiy6//PKZM2d279797LPPrnjVpk2bRo4cma/BAACqUs0Ou9WrV995553Dhg0rKSnp\n16/fXXfdVWmDwsLCs846q/zi4sWLK36WFgAgkpoadtls9p577rnppptWrVrVpUuX2267rWPH\njttu1rBhw0mTJpVfHDFiRL9+/apwTACAqlMjw27z5s0XXnjh+PHjTzjhhLvvvvvUU0/N90QA\nAPlXI8Nu8ODB48eP79u37wMPPFCnTp18jwMAUC1Ux6872bmtW7eOHDnykEMOuf/++1UdAEC5\nmhd2f/3rX5ctW7Zo0aK6detmtjF48OB8DwgAkB81L+wWLlyY7xEAAKqj6v4euyeffLLSyle+\n8pVsNpuXYQAAqrOad8YOAIDtEnYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHs\nAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQd\nAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLAD\nAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYA\nAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQhfke\nAACgBhs/fvy8efOKi4t79OiR71mcsQMA+A+MHj164MCBo0ePzvcgKQk7AIAwhB0AQBDCDgAg\nCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAE\nIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAg\nhB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCE\nsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQ\ndgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDC\nDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELY\nAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7\nAIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEH\nABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewA\nAIIozPkeP/jgg+XLl5eUlJx44okppWw2m8lkcn4vAABUkrMzdlu3bv3pT3/6mc985uCDD+7Y\nseNJJ51Utt6yZctf/vKX2Ww2V3cEAMB25eyM3aWXXjpmzJiUUkFBQWlpafn6Bx988K1vfWvm\nzJkPPPBAru4LAIBt5eaM3YQJE8aMGVNUVHTPPfesXr260lWNGjV68MEHJ06cmJP7AgBgu3IT\ndiNHjkwp3XHHHVdfffVee+1V8aquXbveeOONKaX7778/J/cFAMB25SbsXn/99ZRSr169tnvt\nBRdckFL685//nJP7AgBgu3ITdmvWrEkp7bPPPtu9tmXLlimljz76KCf3BQDAduUm7MrSbc6c\nOdu9dtGiRSmlZs2a5eS+AADYrtyE3Ze//OWU0m233bbda4cMGZJSOuOMM3JyXwAAbFduwm7g\nwIFNmzZ94oknzj///MmTJ5ctzp07d+zYsaeddtojjzxSr169H/zgBzm5LwAAtis332N30EEH\nPfvss927dx83bty4cePKFo844oiyH+rVq/fwww8fddRRObkvAAC2K2d/eeLkk0/+y1/+8v3v\nf79169bli02aNOnVq9e0adMuuuiiXN0RAADblcu/FduiRYs777zzzjvvXLVq1cqVKxs2bLj/\n/vsXFOSsHQEA2Ik9Ul1NmzZt27btgQceuOeqbt68eZdccslBBx1Ur1691q1bd+vW7bXXXttD\n9wUAUCPs/hm7c889d1dv8vTTT+/23VX09ttvn3rqqWVfnpdSWrx48eLFiydOnPjII4/06dMn\nJ3cBAFDj7H7Y/fa3v83hHLvk5ptvXrNmzcUXXzx48OCDDz542bJld99997BhwwYOHCjsAIBa\na/fD7le/+lUO59glZd94fPvtt7dq1Sql1Lp166FDh44cOfLjjz/O10gAAHm3+2H3jW98I4dz\n7JKzzz77z3/+c//+/W+55ZZDDz102bJlP/vZz9avX3/hhRfmayQAgLzL5adiq8ytt966YcOG\nX/ziF4899lj5Yrdu3bY9ibhp06aHH364/OKrr75aNRMCAFS9XIbdunXr3nnnnRUrVmzdunW7\nG3zlK1/JyR2tXLlyxowZpaWlFRenTZs2a9aszp07V1zcuHHj17/+9ZzcKQBANZebsNu6dev1\n119/3333bdy4cSebZbPZnNxdjx49Xn/99e7du//oRz86/PDDP/jggxEjRgwfPvyCCy6YM2dO\ns2bNyrfMZDJNmzYtv1hSUrJ+/fqczAAAUN3kJuyGDBkyfPjwlFL9+vUPOOCAnOxzR2bOnPn6\n66+3bdt27NixdevWTSm1bdt22LBh77///rhx45599tlLLrmkfOO99tpr5cqV5RdHjBjRr1+/\nPToeAEC+5Cbsyt7H9u1vf/vuu++uU6dOTva5IwsXLkwpdezYsazqyp188snjxo0ruxYAoBbK\nzV+GeP/991NKN998856uupTSfvvtl1KaOXPmpk2bKq7/5S9/SSkdfPDBe3oAAIDqKTdh16JF\ni5RSpU8z7CEnnnjigQceuGDBgvPPP3/WrFkbNmx4//33b7311tGjRzdq1Khr165VMAMAQDWU\nm7A777zzUkqPP/54Tva2c/Xq1Rs1alRRUdHvfve7Y489tmHDhq1atRo0aFAmk7n33nv33Xff\nKpgBAKAayk3Y/fjHP+7QocPAgQPvvffeFStW5GSfO3HOOee8+eabffv2bdmyZWFhYdOmTf/r\nv/5r8uTJFT82AQBQ2+TmwxNNmjS54YYbevfuffXVV1999dWFhYXbfbPdzr8MZZe0a9du1KhR\nudobAEAAuQm7cePG9e7du/zili1btmzZkpM9AwDwKeUm7H7yk5+klE444YRBgwa1atWq0heR\nAABQBXITdnPnzk0p/frXvz7iiCNyskMAAHZVbj48UfZZ1D39NycAANiJ3ITdV7/61ZTS888/\nn5O9AQCwG3ITdjfeeOOJJ5547bXXzpgxIyc7BABgV+XmPXbXXXfdoYceOn369E6dOh1zzDHt\n2rVr2LDhtpuV/UlZAAD2hNyE3YMPPlj+86xZs2bNmrXdzYQdAMCek5uwu+2223KyHwAAdltu\nwm7gwIE52Q8AALstNx+eAAAg74QdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEEI\nOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhh\nBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHs\nAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AHAp7VgwYJ33nknpTRjxoz7779/\ny5Yt+Z4I/o2wA4BP5d577z3yyCPnzZuXUlq8ePFVV111zDHHLF68ON9zwf8TdgDwyd58881r\nrrlm8+bNZRez2WxKae7cuX369MnrXPBvhB0AfLIHH3ww/avnypWWlr788svz58/P01BQmbAD\ngE82b968TCaz3avmzJlTxcPAjgg7APhkDRs23FHYNWzYsIqHgR0RdgDwyTp37rx169ZKi5lM\npqio6MQTT8zLSLAtYQcAn+zKK6/8zGc+U1Dw///dLCgoyGazt9xyy1577ZXHwaAiYQcAn6xx\n48ZTp04999xzy1eaNGnyi1/8YsCAAXmcCioRdgDwqbRs2XLcuHFdunRJKX3xi19csmTJN7/5\nzR298Q7yQtgBwC5o0KBBSqlp06Y+M0E1JOwAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCE\nsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQ\ndgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDC\nDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELY\nAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7\nAIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEH\nABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewA\nAHbTunXrVqxYkVJavXr11q1b8z2OsAMA2C1jxow59NBDp06dmlKaMmXKcccd99prr+V3JGEH\nALDLnnjiid69e//jH/8oX5k9e/YZZ5wxZ86cPE4l7AAAdtkNN9xQUFBQWlpavrJ169aSkpIh\nQ4bkcarCPN43AEBN9OGHH7777rvbrpeWlr788stVP085Z+wAAHZNSUnJjq7asGFDVU5SibAD\nANg1Bx54YJMmTbZdLygoOProo6t+nv8fII/3DQBQE9WtW/e///u/t10vLS295pprqn6ecsIO\nAGCXDR48+Lzzzqu4UqdOnVtuuaVnz575GikJOwCA3VBUVDRu3LgXXnjhsMMOSym1a9furbfe\nGjRoUH6nEnYAALvp7LPPLntTXfv27Tt06JDvcYQdAEAUwg4AIAhhBwAQhLADAAhC2AEABCHs\nAACCEHYAAEEIOwCAIIQdAEAQhfkeAIA0fvz4efPmFRcX9+jRI9+zADWYM3YA+Td69OiBAweO\nHj0634MANZuwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgB\nAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAgogQdk8++WRB\nQcGVV16Z70EAAPKpxofd9OnTL7300mw2m+9BAADyrGaH3ZIlS7p3796kSZN8DwIAkH81OOzW\nr1/frVu3unXr3nffffmeBQAg/wrzPcBuymazl1566fz58//0pz+tW7duR5tt3br1zTffLL+4\nePHiKpkOACAPamrY3XjjjePHj//tb3979NFHv/baazvabN26dZ06darKwQAA8qVGht1vfvOb\nIUOG/OxnP/vKV76S71kAAKqLmhd206ZNu/LKK6+88srvfve7n7hxo0aNpk+fXn7x8ccfv+uu\nu/bkdAAAeVPzPjzxwgsvlJSUPPDAA5l/OeWUU1JKDz74YCaT6dKlS8WN69Spc3wFrVq1ytPU\nAAB7XM0LOwAAtqvmhd2NN96Y/XczZ85MKV1xxRXZbPb555/P94AAAPlR88IOAIDtEnYAAEEI\nOwCAIGre151sq2PHjtlsNt9TAADkmTN2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgB\nAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsA\ngCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcA\nEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAAQh7AAA\ngijM9wAAUJP06NGjuLi4Q4cO+R4EtkPYAcAu6NOnT75HgB3yUiwAQBDCDgAgCGEHABCEsAMA\nCELYAQAEIewA8uy1115bsGBBSmnFihX5ngWo2YQdQN4sXbq0S5cup5xyyltvvZVSmjp16lln\nnfX+++/ney6gphJ2APlRWlrarVu3SZMmVVycMmXKl7/85S1btuRrKqBGE3YA+fHiiy9Onz69\ntLS04mJpaemsWbOee+65fE0F1GjCDiA/pk+fvqOr3njjjaqcBAhD2AHkRzab3dFVlU7jAXxK\nwg4gPzp27Lijq4477riqnAQIQ9gB5Mc555zTvn37goJ/Ow4XFBQcfvjhXbt2zddUQI0m7ADy\no7CwcOLEiZ06daq42LFjx2eeeaZevXr5mgqo0YQdQN585jOfee211yZOnHjkkUemlE455ZQ3\n3njj8MMPz/dcQE0l7ADyKZPJfPnLXz7iiCNSSgcccEClV2YBdokjCABAEMIOACAIYQcAEISw\nAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAIQtgBAARRmO8BAABq\nsB49ehQXF3fo0CHfg6Qk7KAWGj9+/Lx584qLi3v06JHvWQBqvD59+uR7hP/npViodUaPHj1w\n4MDRo0fnexAAckzYAQAEIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAE\nIewAAIIQdgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAg\nhB0AQBDCDgAgCGEHABBEYb4HCGj8+PHz5s0rLi7u0aNHvmcBAGoRZ+xyb/To0QMHDhw9enS+\nBwEAahdhBwAQhLADAAhC2AEABCHsAACC8KlYgPzr0aNHcXFxhw4d8j0IULMJO4D869OnT75H\nACLwUizULm+99dbbb7+dUpo5c+ZTTz2V73EAyCVhB7XITTfd9NnPfnb+/PkppUWLFvXs2fPM\nM89cu3ZtvucCIDeEHdQWTz/99ODBg0tLS8suZrPZlNLkyZO/973v5XUuAHJG2EFtMXLkyIKC\n7fyTf+SRRzZs2FD18wCQc8IOaovZs2eXn66rqKSkZOHChVU+DgC5J+ygtmjYsOFuXAVADSLs\noLY4/fTTM5lMpcWCgoLWrVu3bt06LyMBkFvCDmqL/v37N2nSpOLb7AoKCrLZ7J133rlt8AFQ\nEwk7qC0OOeSQP/7xjyeffHL5SosWLR577LELL7wwj1MBkEPCDmqRo48++pVXXjn77LNTSqef\nfvrChQt79eqV76EAyBlhB7VLJpNp3LhxSmmfffapX79+vscBIJeEHQBAEMIOACAIYQcAEISw\nAwAIQtgBAAQh7AAAghB2ubd169aU0oYNG/I9CABQuwi7XProo4+uuOKK3/3udyml559/vm3b\ntmU/AwBUAWGXM+vWrTvttNNGjRqVzWbLVhYuXNi9e/dHH300v4MBALWEsMuZX/3qV/PmzSuv\nupRSaWlpJpO57rrrtmzZksfBAIBaQtjlzKRJkwoKKv8+S0tLly9fPmvWrLyMBADUKsIuZ9as\nWVPxdF1FH3/8cRUPAwDUQoX5HiAP1q9f36tXr5TSBRdccMEFF5Qtjh07duzYsWU/7976oYce\nOm3atO22Xdu2bf/z/Vu3nqv1adOmpZSWLFmS/qV6zmndunXr1ne0viO1Mew2b95c9jtq3759\n+S/onXfeKf/F7d76pZdeOmbMmEr3VVBQ8IUvfKF169YPPfTQf7h/69Zzu75mzZr0L9VhHuvW\nrVu3/unXd6Q2hl3dunXPPffclFKHDh3KFzt06FD+y9q99XPOOad///533XVXxZN2rVq1euih\nh3Kyf+vWc7X+6quvLlmypEmTJtVkHuvWrVu3vqvrO5LZ0dvCQhoxYkS/fv323nvv1atX76G7\nePXVV7/61a8uXrx4//33/+53v/vtb3+7QYMGe+i+YPecd95548eP79Gjx1NPPZXvWQDIJR+e\nyLHPfe5znTp1Kvuhf//+qg4AqDK18aVYqOV69OhRXFz8aU7pA1CzCDuodfr06ZPvEQDYI7wU\nCwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhH7nkg8AAA/6SURB\nVLADAAjCnxTLPX+IEwDIC2GXe/4QJwCQF16KBQAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBA\nEMIOACAIYQcAEISwAwAIQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACAIYQcAEISwAwAI\nQtgBAAQh7AAAghB2AABBCDsAgCCEHQBAEMIOACCImhp269evHzRoUHFxcYMGDVq2bHn55Ze/\n//77+R4KACCfCvM9wO7YvHlz165dJ0+eXHZxyZIlo0aNmjhx4htvvHHIIYfkdzYAgHypkWfs\nRo4cOXny5FatWk2ePHnjxo3vvvtu9+7dly9ffs011+R7NACAvKmRYXfPPfeklB555JHTTz+9\nfv36hx566G9+85tGjRo999xzS5cuzfd0AAD5UfPCbtmyZbNnz27VqtXpp59evti4ceOzzjor\nm82+8soreZwNACCPat577ObOnZtS6tixY6X1o4466re//e17771XcXHt2rUV33VXUlJSBRMC\nAORFzQu7lStXppT222+/Suv77rtvSmnt2rUVF7PZ7KpVq6psNgCAPKp5Ybdp06btrmcymZRS\no0aNKi4WFRXdd9995RenTJny2GOP7dHxAADypeaF3d57753+dd6uorIzcy1atKi4WK9evauu\nuqr8Ytl33W3evHns2LF7fFAAgJwqLi4+9thjd7ZFtqaZP39+SqlNmzaV1i+44IKU0ssvv7yT\n255//vlV83sHAMi56667bueZVPM+FXvYYYe1bNly4cKF06ZNK19cuXLl73//++bNm5966ql5\nnA0AII9q3kuxKaWrrrrq5ptvvvjii3/9618ff/zxc+fOvfrqq9esWfOd73ynTp06O7mhV2DZ\nDYMHD77ppptat269aNGifM9CDbNu3brGjRunlJ588smePXvmexxqhalTp3bu3Dml9N5777Vp\n0ybf41DVamTY9e/f/3e/+90bb7xR8fzckUceeeutt+ZxKgCA/Kp5L8WmlOrXrz9lypT+/fu3\nadOmXr16rVu37tev35/+9KcmTZrkezQAgLypkWfsUkqNGjW644477rjjjnwPAgBQXdTUsIMq\n0759+wsuuKDsG7BhlxQWFpZ9YL9ly5b5noXaYr/99it71lX6YldqiUw2m833DAAA5ECNfI8d\nAADbEnYAAEEIOwCAIIQdAEAQwg4AIAhhR62zfv36QYMGFRcXN2jQoGXLlpdffvn777+fq5tM\nmDAhk8m89NJLuZ+b6mFPPH8mTZp0zjnnNG/evKioqE2bNt27d//DH/6wJx8ENcluPOXK7eiI\n9J/sk+ouC7XJpk2bzjjjjEr/Cvbbb7+FCxfm5CbnnntuSmnKlCl78DGQP3vi+fPEE09kMplt\nD84jR46sqodF9bUbT7mKtntE+g/3STUn7KhdfvGLX6SUWrVqNXny5I0bN7777rvdu3dPKX3p\nS1/6T26ycuXKyZMnX3bZZWWHSGEX1Z54/pT9mfYhQ4YsWbJk06ZNCxYs+M53vpNSKi4urqqH\nRfW1G0+57CcdkXZvn9QUwo7apX379imlyZMnl6+sXbu2UaNGmUzmgw8+2L2bbNiwodL/+wq7\nqHL+/Fm6dGlK6cwzz6x4k9LS0oYNGx500EF76FFQg+zGU+4Tj0i7sU9qEO+xoxZZtmzZ7Nmz\nW7Vqdfrpp5cvNm7c+Kyzzspms6+88sru3aSoqKj8X1TPnj2r4IGQF3vi+XPAAQdks9kXX3yx\n7KqtW7cuWrTohhtuWL9+fbdu3fb0I6Ka242nXPqkI9Lu7ZMaRNhRi8ydOzel1LFjx0rrRx11\nVErpvffey8lNiGpPP3++8Y1vFBYWtmnT5vbbb+/du/fQoUNzNzs10p44/jimhSfsqEVWrlyZ\nUtpvv/0qre+7774ppbVr1+bkJkRVlc+fJ554YtSoUf/ZvNR4e+L445gWnrCjFtm0adN218s+\nk9ioUaOc3ISo9vTz5957712/fv2cOXMGDBiwefPmfv36vfvuu//p0NRke+L445gWnrCjFtl7\n773Tv/6HtaJVq1allFq0aJGTmxBVFTx/GjRo0K5du9tvv/3CCy/cunXrc889l6PZqZH2xPHH\nMS08YUctcthhh6WU3nzzzUrrs2fPLr/2P78JUe2J58/gwYMzmcxPf/rTSht06NAhpbR69erc\njE7NtCeOP45p4Qk7apHDDjusZcuWCxcunDZtWvniypUrf//73zdv3vzUU0/NyU2Iak88f444\n4oiU0rZ/Z+KPf/xj8l/ZWm9PHH8c08ITdtQuV111VUrp4osvfvXVV0tKSmbNmtW1a9c1a9b0\n6dOnTp06uboJUeX8+dOlS5e99977+eef/+EPf7hkyZKSkpLZs2f37dt30qRJzZo1+9KXvlSl\nD4/qZ08cfxzTgqvSb82DfNu4ceMJJ5xQ6V/BkUce+fHHH5dtMGXKlJRSu3btPv1NKir71ihf\nUBzVnnj+PPzww9v+SbF69epNmDAhD4+QamY3nnIVbfeItEvHNGocZ+yoXerXrz9lypT+/fu3\nadOmXr16rVu37tev35/+9KcmTZrk8CZEtSeeP5dddtkzzzxz5pln7rPPPnXr1j3ooIMuuuii\nadOmde3ataoeFtXXnjj+OKbFlslms/meAQCAHHDGDgAgCGEHABCEsAMACELYAQAEIewAAIIQ\ndgAAQQg7AIAghB0AQBDCDgAgCGEHABCEsAMACELYAQAEIewAAIIQdgAAQQg7AIAghB3Ap/X0\n009nMplzzz237OLtt9+eyWTOP//8PXFfffv2zWQy9957784326MzADWOsAMACELYAeymgQMH\nZrPZJ598spbPAFQfwg4AIAhhB9QwZe8qu/LKK9euXTtw4MBDDjmkUaNGxx9//FNPPZVSWrt2\n7Xe/+93WrVsXFRUddthhgwcP3rp1a6U9LF269Nprrz3ssMOKiopatGjRtWvXCRMmbHtHY8eO\nPe2005o0abLPPvuceeaZL7300nYnqfT+tmeeeaZr166tW7du0KDBoYce2r1790mTJm17q4ED\nB5aUlPzoRz9q165dUVHRgQceeNllly1dunTbMebPn//Vr3513333bdiwYceOHYcOHbp58+Yd\nzbCrOweiyQLUKLfddltK6aKLLjrxxBMrHs0ymcyjjz561FFHVTrKXXvttRVv/r//+79Nmzbd\n9mDYu3fvLVu2lG/2wx/+sNIGhYWFV111VUqpe/fuFSfp2bNn+a2uvfba7R5pK85Qdqtrrrmm\n0vwppcMPP3zdunVlm1122WUppa997WuNGjWqtNkZZ5xRUlKy3Rk+5c6BqJyxA2qkxx9/fOHC\nhWPGjFm1atVf//rXTp06ZbPZ3r17v/vuu8OGDVu2bNny5cu//e1vp5Tuu+++devWld3qo48+\n6t69+6pVq3r16jVz5sz169d/8MEHw4cPLyoqevTRR+++++6yzSZPnvyTn/wkpXTVVVf97W9/\nKykpef31148//viRI0fuZKQFCxYMHz68oKBgyJAhixYtKikp+fvf/37bbbcVFBQMHz589uzZ\nFTe+7777Zs+eff/9969atWr9+vUPPPBAQUHB/Pnzx48fX3GzUaNGHXjggRMmTFi3bt3y5cvv\nuuuuOnXqTJ48efDgwTuZ5FPuHAgo32UJsGvKTkqllKZPn16++Mwzz5Qt3nPPPeWLW7ZsqV+/\nfkrp9ddfL1v5/ve/n/79HFuZe+65J6XUpk2bsoudO3dOKfXt27fiNmvXrm3dunXa8Rm7Rx99\nNKV07rnnVtr55z//+ZTSY489Vmn+SZMmVdys7FtUvvnNb5ZdLDtjt++++y5btqziZgMGDEgp\nNWvWrOz84nbP2H3izoGonLEDaqRjjjnm+OOPL7/YqlWrsh969epVvlinTp0DDjggpbRixYqy\nlccffzyldM0111Ta2yWXXFJQULBw4cKlS5euWLFi6tSpKaUf/OAHFbdp3LjxlVdeuZORLr74\n4mw2W+ms2OrVq5csWZJSqvjGuJTSZz/72bPOOqviStnD+cc//lFxsW/fvvvvv3/Fleuuuy6T\nyaxcufKdd97Z0SSfcudAPIX5HgBgd7Rr167ixbp166aUCgsLmzdvvu3GZZ+fWLVq1d///veU\n0plnnrmj3X700UcrVqzIZrNNmzYtLi6udO1xxx33iYO98847L7744jvvvLNw4cJFixa99957\nlZKuzOGHH15ppXHjximljRs37vwe999//wMOOGDp0qWLFy8+5phjtjvDp9w5EI+wA2qkhg0b\nbrtYp06dndxk5cqVn7jbkpKSVatWpZT22muvba8ty6Md+ec///n1r399zJgx5SuFhYXdunX7\n29/+NmvWrEobFxUVfeIwaQcPs1mzZkuXLi0pKdnRrT7lzoF4hB1QW5RH0pw5cyqd8Kuo7HXb\nDz/8MJvNZjKZileVNd+OfOc73xkzZkydOnUuvfTSzp07FxcXH3300Y0bN670qugu2W6MfvDB\nBymlZs2a7fZugai8xw6oLVq0aNGkSZOU0rbnzyoqewW2pKTk7bffrnTVtGnTdnLDsjfwjRgx\n4qGHHrrssstOOeWUsjN8H3744W7PPGPGjEor77333qpVqzKZzI5ehwVqM2EH1BZ16tTp0qVL\nSunnP/95pasmTJiQyWQOPvjglFLbtm3LzucNHTq04jYrV6586KGHdrTzTZs2/fOf/0wpderU\nqeL6lClTtg3ET++xxx4r/+RHmeHDh6eUOnfu7IwdsC1hB9QiN9xwQ7169V555ZWvfe1r8+fP\n37Rp04IFC+64446LL7647NqyzW666aaU0sMPPzxgwIC///3vmzZtev31188+++yydNuuevXq\nlZ3qGzBgwF//+teSkpK5c+f+6Ec/6tq1a7169VJKH3300ZYtW3Z14FWrVp111llTp05dv379\n8uXLb7vtthEjRqSUbrzxxt36BQDBCTugFjn22GMffvjhunXrPvzww8XFxfXr12/btu3AgQPX\nrVt3xRVXlH8NSu/evcu+3PjOO+885JBD6tevf/LJJ8+ZM2fYsGE72fmQIUMymcyUKVPat29f\nVFR0xBFH3HLLLeedd15ZL15//fVl5wt3yYABA+bMmdO5c+dGjRq1aNHihhtuKC0tvfHGG3fy\nwV6gNhN2QO1y0UUXzZgx46KLLtp///3r1at38MEHl/2t2AceeKDiRyV+/vOfP/roo506dSoq\nKmrevHnPnj2nTZt2wgkn7GTPPXv2nDhx4sknn1y/fv2mTZueeeaZTz311K9//etvfetbn//8\n5+vWrVv2Uu8uOfXUU19++eWzzz5777333muvvU477bRx48b9+Mc/3p1HDtQCmWw2m+8ZAADI\nAWfsAACCEHYAAEEIOwCAIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEEIOwCA\nIIQdAEAQwg4AIAhhBwAQhLADAAhC2AEABCHsAACCEHYAAEH8H0HI4UZ0KGrHAAAAAElFTkSu\nQmCC",
      "text/plain": [
       "plot without title"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "dfdnds %>%\n",
    "filter(name == \"wmis\") %>%\n",
    "ggplot(aes(y = mle, ymin = cilow, ymax = cihigh, x = medianbin)) +\n",
    "geom_point() +\n",
    "geom_hline(yintercept = 1.0, lty = 2) +\n",
    "geom_linerange() +\n",
    "scale_x_log10()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [],
   "source": [
    "write_csv(dfdnds, \"FinalFigures/data_for_figures/PD31182_bins.csv\")\n",
    "write_csv(dfdnds.genes, \"FinalFigures/data_for_figures/PD31182_bins_genes.csv\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2 Skin data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now I'll do the same analysis using data from [Martincorena et al. Science 2015](http://science.sciencemag.org/content/348/6237/880/tab-article-info)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Parsed with column specification:\n",
      "cols(\n",
      "  patient = col_character(),\n",
      "  chr = col_character(),\n",
      "  pos = col_integer(),\n",
      "  ref_nt = col_character(),\n",
      "  mut_nt = col_character(),\n",
      "  sampleID = col_character(),\n",
      "  vaf = col_double(),\n",
      "  size = col_double(),\n",
      "  clone.area = col_double(),\n",
      "  gene_name = col_character(),\n",
      "  impact = col_character(),\n",
      "  mutid = col_character()\n",
      ")\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'ADAM29'</li>\n",
       "\t<li>'ADAMTS18'</li>\n",
       "\t<li>'AJUBA'</li>\n",
       "\t<li>'AKT1'</li>\n",
       "\t<li>'AKT2'</li>\n",
       "\t<li>'APOB'</li>\n",
       "\t<li>'ARID1A'</li>\n",
       "\t<li>'ARID2'</li>\n",
       "\t<li>'AURKA'</li>\n",
       "\t<li>'BAI3'</li>\n",
       "\t<li>'BRAF'</li>\n",
       "\t<li>'CASP8'</li>\n",
       "\t<li>'CCND1'</li>\n",
       "\t<li>'CDH1'</li>\n",
       "\t<li>'CDKN2A'</li>\n",
       "\t<li>'CR2'</li>\n",
       "\t<li>'CREBBP'</li>\n",
       "\t<li>'CUL3'</li>\n",
       "\t<li>'DICER1'</li>\n",
       "\t<li>'EGFR'</li>\n",
       "\t<li>'EPHA2'</li>\n",
       "\t<li>'ERBB2'</li>\n",
       "\t<li>'ERBB3'</li>\n",
       "\t<li>'ERBB4'</li>\n",
       "\t<li>'EZH2'</li>\n",
       "\t<li>'FAT1'</li>\n",
       "\t<li>'FAT4'</li>\n",
       "\t<li>'FBXW7'</li>\n",
       "\t<li>'FGFR1'</li>\n",
       "\t<li>'FGFR2'</li>\n",
       "\t<li>'FGFR3'</li>\n",
       "\t<li>'FLG2'</li>\n",
       "\t<li>'GRIN2A'</li>\n",
       "\t<li>'GRM3'</li>\n",
       "\t<li>'HRAS'</li>\n",
       "\t<li>'IRF6'</li>\n",
       "\t<li>'KCNH5'</li>\n",
       "\t<li>'KEAP1'</li>\n",
       "\t<li>'KMT2A'</li>\n",
       "\t<li>'KMT2C'</li>\n",
       "\t<li>'KMT2D'</li>\n",
       "\t<li>'KRAS'</li>\n",
       "\t<li>'MET'</li>\n",
       "\t<li>'MUC17'</li>\n",
       "\t<li>'NF1'</li>\n",
       "\t<li>'NFE2L2'</li>\n",
       "\t<li>'NOTCH1'</li>\n",
       "\t<li>'NOTCH2'</li>\n",
       "\t<li>'NOTCH3'</li>\n",
       "\t<li>'NOTCH4'</li>\n",
       "\t<li>'NRAS'</li>\n",
       "\t<li>'NSD1'</li>\n",
       "\t<li>'PCED1B'</li>\n",
       "\t<li>'PIK3CA'</li>\n",
       "\t<li>'PLCB1'</li>\n",
       "\t<li>'PPP1R3A'</li>\n",
       "\t<li>'PREX2'</li>\n",
       "\t<li>'PTCH1'</li>\n",
       "\t<li>'PTEN'</li>\n",
       "\t<li>'PTPRT'</li>\n",
       "\t<li>'RB1'</li>\n",
       "\t<li>'RBM10'</li>\n",
       "\t<li>'SALL1'</li>\n",
       "\t<li>'SCN11A'</li>\n",
       "\t<li>'SCN1A'</li>\n",
       "\t<li>'SETD2'</li>\n",
       "\t<li>'SMAD4'</li>\n",
       "\t<li>'SMO'</li>\n",
       "\t<li>'SOX2'</li>\n",
       "\t<li>'SPHKAP'</li>\n",
       "\t<li>'SUFU'</li>\n",
       "\t<li>'TP53'</li>\n",
       "\t<li>'TP63'</li>\n",
       "\t<li>'TRIOBP'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 'ADAM29'\n",
       "\\item 'ADAMTS18'\n",
       "\\item 'AJUBA'\n",
       "\\item 'AKT1'\n",
       "\\item 'AKT2'\n",
       "\\item 'APOB'\n",
       "\\item 'ARID1A'\n",
       "\\item 'ARID2'\n",
       "\\item 'AURKA'\n",
       "\\item 'BAI3'\n",
       "\\item 'BRAF'\n",
       "\\item 'CASP8'\n",
       "\\item 'CCND1'\n",
       "\\item 'CDH1'\n",
       "\\item 'CDKN2A'\n",
       "\\item 'CR2'\n",
       "\\item 'CREBBP'\n",
       "\\item 'CUL3'\n",
       "\\item 'DICER1'\n",
       "\\item 'EGFR'\n",
       "\\item 'EPHA2'\n",
       "\\item 'ERBB2'\n",
       "\\item 'ERBB3'\n",
       "\\item 'ERBB4'\n",
       "\\item 'EZH2'\n",
       "\\item 'FAT1'\n",
       "\\item 'FAT4'\n",
       "\\item 'FBXW7'\n",
       "\\item 'FGFR1'\n",
       "\\item 'FGFR2'\n",
       "\\item 'FGFR3'\n",
       "\\item 'FLG2'\n",
       "\\item 'GRIN2A'\n",
       "\\item 'GRM3'\n",
       "\\item 'HRAS'\n",
       "\\item 'IRF6'\n",
       "\\item 'KCNH5'\n",
       "\\item 'KEAP1'\n",
       "\\item 'KMT2A'\n",
       "\\item 'KMT2C'\n",
       "\\item 'KMT2D'\n",
       "\\item 'KRAS'\n",
       "\\item 'MET'\n",
       "\\item 'MUC17'\n",
       "\\item 'NF1'\n",
       "\\item 'NFE2L2'\n",
       "\\item 'NOTCH1'\n",
       "\\item 'NOTCH2'\n",
       "\\item 'NOTCH3'\n",
       "\\item 'NOTCH4'\n",
       "\\item 'NRAS'\n",
       "\\item 'NSD1'\n",
       "\\item 'PCED1B'\n",
       "\\item 'PIK3CA'\n",
       "\\item 'PLCB1'\n",
       "\\item 'PPP1R3A'\n",
       "\\item 'PREX2'\n",
       "\\item 'PTCH1'\n",
       "\\item 'PTEN'\n",
       "\\item 'PTPRT'\n",
       "\\item 'RB1'\n",
       "\\item 'RBM10'\n",
       "\\item 'SALL1'\n",
       "\\item 'SCN11A'\n",
       "\\item 'SCN1A'\n",
       "\\item 'SETD2'\n",
       "\\item 'SMAD4'\n",
       "\\item 'SMO'\n",
       "\\item 'SOX2'\n",
       "\\item 'SPHKAP'\n",
       "\\item 'SUFU'\n",
       "\\item 'TP53'\n",
       "\\item 'TP63'\n",
       "\\item 'TRIOBP'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 'ADAM29'\n",
       "2. 'ADAMTS18'\n",
       "3. 'AJUBA'\n",
       "4. 'AKT1'\n",
       "5. 'AKT2'\n",
       "6. 'APOB'\n",
       "7. 'ARID1A'\n",
       "8. 'ARID2'\n",
       "9. 'AURKA'\n",
       "10. 'BAI3'\n",
       "11. 'BRAF'\n",
       "12. 'CASP8'\n",
       "13. 'CCND1'\n",
       "14. 'CDH1'\n",
       "15. 'CDKN2A'\n",
       "16. 'CR2'\n",
       "17. 'CREBBP'\n",
       "18. 'CUL3'\n",
       "19. 'DICER1'\n",
       "20. 'EGFR'\n",
       "21. 'EPHA2'\n",
       "22. 'ERBB2'\n",
       "23. 'ERBB3'\n",
       "24. 'ERBB4'\n",
       "25. 'EZH2'\n",
       "26. 'FAT1'\n",
       "27. 'FAT4'\n",
       "28. 'FBXW7'\n",
       "29. 'FGFR1'\n",
       "30. 'FGFR2'\n",
       "31. 'FGFR3'\n",
       "32. 'FLG2'\n",
       "33. 'GRIN2A'\n",
       "34. 'GRM3'\n",
       "35. 'HRAS'\n",
       "36. 'IRF6'\n",
       "37. 'KCNH5'\n",
       "38. 'KEAP1'\n",
       "39. 'KMT2A'\n",
       "40. 'KMT2C'\n",
       "41. 'KMT2D'\n",
       "42. 'KRAS'\n",
       "43. 'MET'\n",
       "44. 'MUC17'\n",
       "45. 'NF1'\n",
       "46. 'NFE2L2'\n",
       "47. 'NOTCH1'\n",
       "48. 'NOTCH2'\n",
       "49. 'NOTCH3'\n",
       "50. 'NOTCH4'\n",
       "51. 'NRAS'\n",
       "52. 'NSD1'\n",
       "53. 'PCED1B'\n",
       "54. 'PIK3CA'\n",
       "55. 'PLCB1'\n",
       "56. 'PPP1R3A'\n",
       "57. 'PREX2'\n",
       "58. 'PTCH1'\n",
       "59. 'PTEN'\n",
       "60. 'PTPRT'\n",
       "61. 'RB1'\n",
       "62. 'RBM10'\n",
       "63. 'SALL1'\n",
       "64. 'SCN11A'\n",
       "65. 'SCN1A'\n",
       "66. 'SETD2'\n",
       "67. 'SMAD4'\n",
       "68. 'SMO'\n",
       "69. 'SOX2'\n",
       "70. 'SPHKAP'\n",
       "71. 'SUFU'\n",
       "72. 'TP53'\n",
       "73. 'TP63'\n",
       "74. 'TRIOBP'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       " [1] \"ADAM29\"   \"ADAMTS18\" \"AJUBA\"    \"AKT1\"     \"AKT2\"     \"APOB\"    \n",
       " [7] \"ARID1A\"   \"ARID2\"    \"AURKA\"    \"BAI3\"     \"BRAF\"     \"CASP8\"   \n",
       "[13] \"CCND1\"    \"CDH1\"     \"CDKN2A\"   \"CR2\"      \"CREBBP\"   \"CUL3\"    \n",
       "[19] \"DICER1\"   \"EGFR\"     \"EPHA2\"    \"ERBB2\"    \"ERBB3\"    \"ERBB4\"   \n",
       "[25] \"EZH2\"     \"FAT1\"     \"FAT4\"     \"FBXW7\"    \"FGFR1\"    \"FGFR2\"   \n",
       "[31] \"FGFR3\"    \"FLG2\"     \"GRIN2A\"   \"GRM3\"     \"HRAS\"     \"IRF6\"    \n",
       "[37] \"KCNH5\"    \"KEAP1\"    \"KMT2A\"    \"KMT2C\"    \"KMT2D\"    \"KRAS\"    \n",
       "[43] \"MET\"      \"MUC17\"    \"NF1\"      \"NFE2L2\"   \"NOTCH1\"   \"NOTCH2\"  \n",
       "[49] \"NOTCH3\"   \"NOTCH4\"   \"NRAS\"     \"NSD1\"     \"PCED1B\"   \"PIK3CA\"  \n",
       "[55] \"PLCB1\"    \"PPP1R3A\"  \"PREX2\"    \"PTCH1\"    \"PTEN\"     \"PTPRT\"   \n",
       "[61] \"RB1\"      \"RBM10\"    \"SALL1\"    \"SCN11A\"   \"SCN1A\"    \"SETD2\"   \n",
       "[67] \"SMAD4\"    \"SMO\"      \"SOX2\"     \"SPHKAP\"   \"SUFU\"     \"TP53\"    \n",
       "[73] \"TP63\"     \"TRIOBP\"  "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "df <- read_csv(\"data/skin/skin_mutation.csv\")\n",
    "data(\"dataset_normalskin_genes\", package=\"dndscv\")\n",
    "target_genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] 0.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.83).\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.037482 0.7364296 1.461604\n",
      "wnon wnon 1.154040 0.5421908 2.456344\n",
      "wspl wspl 1.423848 0.4096813 4.948585\n",
      "wtru wtru 1.213696 0.6239409 2.360892\n",
      "wall wall 1.051715 0.7506776 1.473474\n",
      "[1] 0.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.07).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2.”    Regression model for indels (theta = 1.25)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.206938 1.053023 1.383350\n",
      "wnon wnon 1.691060 1.289734 2.217266\n",
      "wspl wspl 2.194930 1.505110 3.200906\n",
      "wtru wtru 1.830232 1.451754 2.307380\n",
      "wall wall 1.259358 1.101371 1.440007\n",
      "[1] 0.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.82).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 0.706)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.202285 1.074094 1.345775\n",
      "wnon wnon 1.792658 1.443303 2.226576\n",
      "wspl wspl 2.260949 1.667530 3.065545\n",
      "wtru wtru 1.921952 1.595135 2.315728\n",
      "wall wall 1.264070 1.131626 1.412015\n",
      "[1] 0.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.7).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.05)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.227223 1.106569 1.361033\n",
      "wnon wnon 1.867418 1.537109 2.268706\n",
      "wspl wspl 2.326797 1.766937 3.064050\n",
      "wtru wtru 1.993110 1.684456 2.358321\n",
      "wall wall 1.294020 1.169021 1.432384\n",
      "[1] 0.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.67).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 0.665)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.236855 1.120057 1.365832\n",
      "wnon wnon 1.862398 1.547028 2.242058\n",
      "wspl wspl 2.390166 1.837539 3.108991\n",
      "wtru wtru 2.003521 1.706142 2.352734\n",
      "wall wall 1.303831 1.182856 1.437178\n",
      "[1] 0.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.71).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 0.517)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.239759 1.125700 1.365374\n",
      "wnon wnon 1.858720 1.551278 2.227092\n",
      "wspl wspl 2.454834 1.906919 3.160181\n",
      "wtru wtru 2.018969 1.727180 2.360053\n",
      "wall wall 1.307130 1.188940 1.437070\n",
      "[1] 0.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.65).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.248726 1.135090 1.373737\n",
      "wnon wnon 1.885860 1.579036 2.252304\n",
      "wspl wspl 2.499893 1.949841 3.205117\n",
      "wtru wtru 2.049852 1.758116 2.389998\n",
      "wall wall 1.317899 1.200015 1.447364\n",
      "[1] 0.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.7).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.261816 1.148064 1.386838\n",
      "wnon wnon 1.926434 1.616712 2.295490\n",
      "wspl wspl 2.522860 1.972970 3.226012\n",
      "wtru wtru 2.086140 1.792472 2.427922\n",
      "wall wall 1.332517 1.214419 1.462099\n",
      "[1] 0.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.7).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.267971 1.154448 1.392656\n",
      "wnon wnon 1.927863 1.619939 2.294318\n",
      "wspl wspl 2.594289 2.037945 3.302510\n",
      "wtru wtru 2.106649 1.812751 2.448196\n",
      "wall wall 1.339662 1.221746 1.468958\n",
      "[1] 0.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.7).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.268660 1.155550 1.392841\n",
      "wnon wnon 1.953642 1.643839 2.321830\n",
      "wspl wspl 2.684356 2.117108 3.403591\n",
      "wtru wtru 2.149962 1.852650 2.494987\n",
      "wall wall 1.343649 1.225893 1.472715\n",
      "[1] 0.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.283553 1.169527 1.408698\n",
      "wnon wnon 1.945557 1.637963 2.310914\n",
      "wspl wspl 2.663228 2.100920 3.376038\n",
      "wtru wtru 2.138050 1.843140 2.480146\n",
      "wall wall 1.356651 1.238155 1.486486\n",
      "[1] 0.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.67).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.282333 1.168753 1.406952\n",
      "wnon wnon 1.967059 1.658209 2.333434\n",
      "wspl wspl 2.659314 2.098072 3.370691\n",
      "wtru wtru 2.151855 1.856489 2.494213\n",
      "wall wall 1.356898 1.238748 1.486317\n",
      "[1] 0.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.284564 1.171087 1.409036\n",
      "wnon wnon 1.987411 1.676864 2.355471\n",
      "wspl wspl 2.641227 2.084342 3.346898\n",
      "wtru wtru 2.162449 1.866699 2.505057\n",
      "wall wall 1.359861 1.241763 1.489190\n",
      "[1] 0.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.69).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.280830 1.167909 1.404668\n",
      "wnon wnon 1.984463 1.674985 2.351122\n",
      "wspl wspl 2.626248 2.072837 3.327410\n",
      "wtru wtru 2.156265 1.861832 2.497259\n",
      "wall wall 1.355908 1.238390 1.484578\n",
      "[1] 0.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.69).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.281829 1.168920 1.405644\n",
      "wnon wnon 1.981911 1.673364 2.347350\n",
      "wspl wspl 2.626693 2.073252 3.327871\n",
      "wtru wtru 2.153835 1.860086 2.493973\n",
      "wall wall 1.356726 1.239239 1.485350\n",
      "[1] 0.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.74).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.283292 1.170447 1.407018\n",
      "wnon wnon 1.978879 1.670962 2.343538\n",
      "wspl wspl 2.645536 2.090085 3.348600\n",
      "wtru wtru 2.156719 1.862967 2.496790\n",
      "wall wall 1.358090 1.240679 1.486611\n",
      "[1] 0.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.74).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.524)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.284913 1.171946 1.408769\n",
      "wnon wnon 1.977390 1.669752 2.341707\n",
      "wspl wspl 2.636753 2.083341 3.337171\n",
      "wtru wtru 2.153573 1.860292 2.493091\n",
      "wall wall 1.359342 1.241839 1.487964\n",
      "[1] 0.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.389)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.282834 1.170192 1.406319\n",
      "wnon wnon 1.975038 1.667842 2.338816\n",
      "wspl wspl 2.629147 2.077492 3.327287\n",
      "wtru wtru 2.150044 1.857356 2.488855\n",
      "wall wall 1.357014 1.239858 1.485240\n",
      "[1] 0.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.284488 1.171717 1.408112\n",
      "wnon wnon 1.976069 1.668722 2.340025\n",
      "wspl wspl 2.631297 2.079202 3.329993\n",
      "wtru wtru 2.151334 1.858475 2.490343\n",
      "wall wall 1.358596 1.241317 1.486956\n",
      "[1] 0.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.285283 1.172451 1.408973\n",
      "wnon wnon 1.976141 1.668798 2.340087\n",
      "wspl wspl 2.627241 2.076106 3.324685\n",
      "wtru wtru 2.150502 1.857781 2.489344\n",
      "wall wall 1.359300 1.241969 1.487716\n",
      "[1] 1.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, NOTCH2, FAT1, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.288597 1.175525 1.412546\n",
      "wnon wnon 1.975350 1.668198 2.339057\n",
      "wspl wspl 2.627737 2.076600 3.325147\n",
      "wtru wtru 2.149945 1.857343 2.488643\n",
      "wall wall 1.362235 1.244693 1.490878\n",
      "[1] 1.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.289798 1.176634 1.413845\n",
      "wnon wnon 1.984079 1.675975 2.348824\n",
      "wspl wspl 2.656603 2.101349 3.358577\n",
      "wtru wtru 2.164101 1.870097 2.504327\n",
      "wall wall 1.364580 1.246860 1.493415\n",
      "[1] 1.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.290478 1.177262 1.414580\n",
      "wnon wnon 1.993611 1.684465 2.359493\n",
      "wspl wspl 2.656267 2.101132 3.358072\n",
      "wtru wtru 2.171051 1.876383 2.511994\n",
      "wall wall 1.365877 1.248058 1.494818\n",
      "[1] 1.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291659 1.178348 1.415866\n",
      "wnon wnon 1.994918 1.685563 2.361050\n",
      "wspl wspl 2.658080 2.102565 3.360367\n",
      "wtru wtru 2.172496 1.877624 2.513675\n",
      "wall wall 1.367044 1.249131 1.496087\n",
      "[1] 1.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291659 1.178348 1.415866\n",
      "wnon wnon 1.994918 1.685563 2.361050\n",
      "wspl wspl 2.658080 2.102565 3.360367\n",
      "wtru wtru 2.172496 1.877624 2.513675\n",
      "wall wall 1.367044 1.249131 1.496087\n",
      "[1] 1.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291998 1.178663 1.416230\n",
      "wnon wnon 1.993705 1.684564 2.359579\n",
      "wspl wspl 2.658017 2.102530 3.360262\n",
      "wtru wtru 2.171502 1.876782 2.512503\n",
      "wall wall 1.367277 1.249348 1.496337\n",
      "[1] 1.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291708 1.178398 1.415912\n",
      "wnon wnon 1.993686 1.684548 2.359556\n",
      "wspl wspl 2.678779 2.120711 3.383704\n",
      "wtru wtru 2.177291 1.882034 2.518868\n",
      "wall wall 1.367413 1.249475 1.496483\n",
      "[1] 1.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291708 1.178398 1.415912\n",
      "wnon wnon 1.993686 1.684548 2.359556\n",
      "wspl wspl 2.678779 2.120711 3.383704\n",
      "wtru wtru 2.177291 1.882034 2.518868\n",
      "wall wall 1.367413 1.249475 1.496483\n",
      "[1] 1.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291708 1.178398 1.415912\n",
      "wnon wnon 1.993686 1.684548 2.359556\n",
      "wspl wspl 2.678779 2.120711 3.383704\n",
      "wtru wtru 2.177291 1.882034 2.518868\n",
      "wall wall 1.367413 1.249475 1.496483\n",
      "[1] 1.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291708 1.178398 1.415912\n",
      "wnon wnon 1.993686 1.684548 2.359556\n",
      "wspl wspl 2.678779 2.120711 3.383704\n",
      "wtru wtru 2.177291 1.882034 2.518868\n",
      "wall wall 1.367413 1.249475 1.496483\n",
      "[1] 1.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 1.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 2.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292513 1.179143 1.416783\n",
      "wnon wnon 1.994577 1.685297 2.360614\n",
      "wspl wspl 2.679998 2.121678 3.385241\n",
      "wtru wtru 2.178272 1.882880 2.520007\n",
      "wall wall 1.368202 1.250205 1.497336\n",
      "[1] 2.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n",
      "[1] 2.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53, RBM10.”    Regression model for indels (theta = 0.48)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.292985 1.179582 1.417291\n",
      "wnon wnon 1.994306 1.685085 2.360271\n",
      "wspl wspl 2.680350 2.121968 3.385666\n",
      "wtru wtru 2.178124 1.882764 2.519820\n",
      "wall wall 1.368617 1.250591 1.497782\n"
     ]
    }
   ],
   "source": [
    "minarea <- 0.03\n",
    "maxarea <- 2.5\n",
    "step <- 0.05\n",
    "areacutoff <- seq(minarea, maxarea, step)\n",
    "\n",
    "dfdnds <- data.frame()\n",
    "dfdnds.genes <- data.frame()\n",
    "\n",
    "for (cutoff in areacutoff){\n",
    "  print(cutoff)\n",
    "  x1 <- df %>%\n",
    "    filter(clone.area < cutoff) #filter for mutations with ccf < cutoff\n",
    "  x <- dndscv(x1, gene_list = target_genes,\n",
    "              outp = 3, max_muts_per_gene_per_sample = Inf, \n",
    "              max_coding_muts_per_sample = Inf)\n",
    "  out <- x$globaldnds %>%\n",
    "    mutate(areacutoff = cutoff, nmutations = length(x1$patient))\n",
    "  dfdnds <- rbind(dfdnds, out)\n",
    "  combined <- left_join(x$sel_cv, x$sel_loc, by = c(\"gene_name\", \"n_syn\", \"n_mis\", \"n_non\", \"n_spl\"))\n",
    "  out2 <- combined %>%\n",
    "    mutate(areacutoff = cutoff)\n",
    "  dfdnds.genes <- bind_rows(dfdnds.genes, out2)\n",
    "  print(x$globaldnds)\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "write_csv(dfdnds, \"data/skin/dnds_combined.csv\")\n",
    "write_csv(dfdnds.genes, \"data/skin/dnds_genes_combined.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1] \"PD13634\"\n",
      "[1] 1\n",
      "[1] 0.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.69e+03).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow    cihigh\n",
      "wmis wmis 0.9190661 0.5052408  1.671841\n",
      "wnon wnon 1.3358329 0.4118339  4.332935\n",
      "wspl wspl 2.1834403 0.2623568 18.171482\n",
      "wtru wtru 1.4683526 0.5077709  4.246126\n",
      "wall wall 0.9708999 0.5429471  1.736167\n",
      "[1] 0.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.95).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow   cihigh\n",
      "wmis wmis 0.9959472 0.7620217 1.301683\n",
      "wnon wnon 0.8250718 0.4244530 1.603813\n",
      "wspl wspl 1.4724166 0.6373717 3.401486\n",
      "wtru wtru 0.9996541 0.5837059 1.712007\n",
      "wall wall 0.9944332 0.7640813 1.294231\n",
      "[1] 0.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.76).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.093470 0.8743743 1.367466\n",
      "wnon wnon 1.414162 0.8864907 2.255922\n",
      "wspl wspl 1.324316 0.6618203 2.649985\n",
      "wtru wtru 1.386586 0.9271860 2.073607\n",
      "wall wall 1.120317 0.8992776 1.395687\n",
      "[1] 0.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.71).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.109635 0.9025157 1.364287\n",
      "wnon wnon 1.427183 0.9311144 2.187541\n",
      "wspl wspl 1.369767 0.7259423 2.584587\n",
      "wtru wtru 1.409896 0.9749070 2.038970\n",
      "wall wall 1.137234 0.9282358 1.393290\n",
      "[1] 0.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.65).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.159081 0.9517162 1.411627\n",
      "wnon wnon 1.510567 1.0091897 2.261036\n",
      "wspl wspl 1.397754 0.7587234 2.575003\n",
      "wtru wtru 1.476966 1.0399062 2.097717\n",
      "wall wall 1.187691 0.9782936 1.441908\n",
      "[1] 0.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.83).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.134397 0.9363488 1.374334\n",
      "wnon wnon 1.388878 0.9310222 2.071898\n",
      "wspl wspl 1.417044 0.7848669 2.558412\n",
      "wtru wtru 1.397104 0.9886501 1.974307\n",
      "wall wall 1.157583 0.9583419 1.398247\n",
      "[1] 0.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.126822 0.9320992 1.362223\n",
      "wnon wnon 1.379521 0.9298607 2.046627\n",
      "wspl wspl 1.387474 0.7695618 2.501531\n",
      "wtru wtru 1.381823 0.9815416 1.945342\n",
      "wall wall 1.149467 0.9536408 1.385506\n",
      "[1] 0.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.86).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.107499 0.9183387 1.335624\n",
      "wnon wnon 1.414184 0.9617321 2.079495\n",
      "wspl wspl 1.571302 0.8997498 2.744084\n",
      "wtru wtru 1.459211 1.0471987 2.033326\n",
      "wall wall 1.138265 0.9467953 1.368455\n",
      "[1] 0.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.87).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.104957 0.9172123 1.331130\n",
      "wnon wnon 1.391009 0.9468465 2.043528\n",
      "wspl wspl 1.805948 1.0725137 3.040938\n",
      "wtru wtru 1.510979 1.0915921 2.091492\n",
      "wall wall 1.139744 0.9491051 1.368674\n",
      "[1] 0.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.96).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.100640 0.9151509 1.323726\n",
      "wnon wnon 1.359224 0.9262401 1.994612\n",
      "wspl wspl 1.966300 1.1915047 3.244920\n",
      "wtru wtru 1.533557 1.1128797 2.113255\n",
      "wall wall 1.136909 0.9482922 1.363041\n",
      "[1] 0.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.96).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.119897 0.9316411 1.346195\n",
      "wnon wnon 1.358413 0.9260640 1.992612\n",
      "wspl wspl 1.988735 1.2052120 3.281636\n",
      "wtru wtru 1.537854 1.1162956 2.118610\n",
      "wall wall 1.154773 0.9636209 1.383844\n",
      "[1] 0.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.92).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.124795 0.9363684 1.351139\n",
      "wnon wnon 1.426483 0.9801633 2.076037\n",
      "wspl wspl 1.996898 1.2103730 3.294522\n",
      "wtru wtru 1.587748 1.1572761 2.178341\n",
      "wall wall 1.163805 0.9718801 1.393630\n",
      "[1] 0.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.93).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.125771 0.9377534 1.351486\n",
      "wnon wnon 1.485426 1.0277726 2.146867\n",
      "wspl wspl 1.998909 1.2118448 3.297154\n",
      "wtru wtru 1.629627 1.1922014 2.227546\n",
      "wall wall 1.168587 0.9765154 1.398438\n",
      "[1] 0.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.01).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.121889 0.9349436 1.346215\n",
      "wnon wnon 1.454635 1.0066870 2.101906\n",
      "wspl wspl 1.967816 1.1940138 3.243095\n",
      "wtru wtru 1.598497 1.1697491 2.184394\n",
      "wall wall 1.162591 0.9719235 1.390662\n",
      "[1] 0.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.01).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.121889 0.9349436 1.346215\n",
      "wnon wnon 1.454635 1.0066870 2.101906\n",
      "wspl wspl 1.967816 1.1940138 3.243095\n",
      "wtru wtru 1.598497 1.1697491 2.184394\n",
      "wall wall 1.162591 0.9719235 1.390662\n",
      "[1] 0.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.01).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.128568 0.9406567 1.354019\n",
      "wnon wnon 1.452948 1.0057111 2.099069\n",
      "wspl wspl 1.975307 1.1986012 3.255325\n",
      "wtru wtru 1.598831 1.1701367 2.184583\n",
      "wall wall 1.168731 0.9771902 1.397816\n",
      "[1] 0.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.01).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.132649 0.9441571 1.358772\n",
      "wnon wnon 1.450894 1.0043859 2.095900\n",
      "wspl wspl 1.947998 1.1829755 3.207757\n",
      "wtru wtru 1.590943 1.1644792 2.173588\n",
      "wall wall 1.171839 0.9798549 1.401438\n",
      "[1] 0.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.127297 0.9400212 1.351882\n",
      "wnon wnon 1.446150 1.0012405 2.088758\n",
      "wspl wspl 1.941002 1.1788501 3.195900\n",
      "wtru wtru 1.585589 1.1607480 2.165925\n",
      "wall wall 1.166405 0.9756504 1.394455\n",
      "[1] 0.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.127297 0.9400212 1.351882\n",
      "wnon wnon 1.446150 1.0012405 2.088758\n",
      "wspl wspl 1.941002 1.1788501 3.195900\n",
      "wtru wtru 1.585589 1.1607480 2.165925\n",
      "wall wall 1.166405 0.9756504 1.394455\n",
      "[1] 0.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.127297 0.9400212 1.351882\n",
      "wnon wnon 1.446150 1.0012405 2.088758\n",
      "wspl wspl 1.941002 1.1788501 3.195900\n",
      "wtru wtru 1.585589 1.1607480 2.165925\n",
      "wall wall 1.166405 0.9756504 1.394455\n",
      "[1] 1.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 1.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] 2.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.129084 0.9415724 1.353938\n",
      "wnon wnon 1.445385 1.0007872 2.087494\n",
      "wspl wspl 1.942178 1.1796178 3.197694\n",
      "wtru wtru 1.585240 1.1605532 2.165335\n",
      "wall wall 1.167994 0.9770319 1.396280\n",
      "[1] \"PD18003\"\n",
      "[1] 1\n",
      "[1] 0.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 7.15).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow    cihigh\n",
      "wmis wmis 0.9360599 0.5075689  1.726284\n",
      "wnon wnon 1.2843155 0.3380496  4.879362\n",
      "wspl wspl 3.0813833 0.6024808 15.759711\n",
      "wtru wtru 1.7092459 0.5762137  5.070205\n",
      "wall wall 0.9936811 0.5453045  1.810735\n",
      "[1] 0.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.18).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1.”    Regression model for indels (theta = 0.49)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.255639 1.008270 1.563697\n",
      "wnon wnon 2.204453 1.483167 3.276512\n",
      "wspl wspl 2.597983 1.489488 4.531432\n",
      "wtru wtru 2.316459 1.643118 3.265731\n",
      "wall wall 1.348217 1.087291 1.671759\n",
      "[1] 0.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.05).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1.”    Regression model for indels (theta = 12.8)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.207515 1.011798 1.441091\n",
      "wnon wnon 1.919301 1.387313 2.655288\n",
      "wspl wspl 2.762490 1.767532 4.317518\n",
      "wtru wtru 2.140825 1.618899 2.831017\n",
      "wall wall 1.289882 1.084714 1.533856\n",
      "[1] 0.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.88).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 22.1)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.246372 1.062201 1.462475\n",
      "wnon wnon 2.081273 1.563427 2.770642\n",
      "wspl wspl 2.787769 1.871179 4.153349\n",
      "wtru wtru 2.269484 1.770695 2.908778\n",
      "wall wall 1.337710 1.143753 1.564559\n",
      "[1] 0.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.8).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 0.78)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.265967 1.086567 1.474987\n",
      "wnon wnon 1.981741 1.505614 2.608436\n",
      "wspl wspl 2.956413 2.024992 4.316254\n",
      "wtru wtru 2.230122 1.758159 2.828779\n",
      "wall wall 1.352611 1.164490 1.571122\n",
      "[1] 0.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.73).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.271856 1.097155 1.474374\n",
      "wnon wnon 2.011338 1.544134 2.619903\n",
      "wspl wspl 2.969597 2.069217 4.261760\n",
      "wtru wtru 2.259560 1.797828 2.839878\n",
      "wall wall 1.360183 1.176761 1.572194\n",
      "[1] 0.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.67).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.291341 1.116111 1.494082\n",
      "wnon wnon 2.065516 1.593384 2.677545\n",
      "wspl wspl 3.047545 2.139830 4.340314\n",
      "wtru wtru 2.320826 1.854136 2.904982\n",
      "wall wall 1.382437 1.198194 1.595010\n",
      "[1] 0.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.66).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.330508 1.151652 1.537141\n",
      "wnon wnon 2.120924 1.640084 2.742736\n",
      "wspl wspl 3.002865 2.109787 4.273986\n",
      "wtru wtru 2.352553 1.882414 2.940112\n",
      "wall wall 1.419775 1.232167 1.635946\n",
      "[1] 0.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.61).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.334663 1.156644 1.540080\n",
      "wnon wnon 2.108248 1.634252 2.719722\n",
      "wspl wspl 3.026801 2.135080 4.290950\n",
      "wtru wtru 2.348083 1.882823 2.928313\n",
      "wall wall 1.423481 1.236841 1.638285\n",
      "[1] 0.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.6).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.339824 1.161580 1.545418\n",
      "wnon wnon 2.194040 1.706420 2.820999\n",
      "wspl wspl 3.157093 2.239320 4.451011\n",
      "wtru wtru 2.446027 1.966693 3.042187\n",
      "wall wall 1.436566 1.248770 1.652605\n",
      "[1] 0.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.350504 1.171905 1.556321\n",
      "wnon wnon 2.172144 1.692171 2.788257\n",
      "wspl wspl 3.086335 2.190076 4.349374\n",
      "wtru wtru 2.410999 1.940858 2.995024\n",
      "wall wall 1.444250 1.256536 1.660006\n",
      "[1] 0.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.55).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.350343 1.172094 1.555700\n",
      "wnon wnon 2.197784 1.714893 2.816652\n",
      "wspl wspl 3.086421 2.190343 4.349089\n",
      "wtru wtru 2.429012 1.957194 3.014570\n",
      "wall wall 1.445923 1.258359 1.661445\n",
      "[1] 0.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.53).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.358882 1.179989 1.564896\n",
      "wnon wnon 2.220295 1.733802 2.843296\n",
      "wspl wspl 3.066867 2.177680 4.319126\n",
      "wtru wtru 2.442132 1.968905 3.029100\n",
      "wall wall 1.454717 1.266496 1.670910\n",
      "[1] 0.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.6).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.357382 1.179006 1.562745\n",
      "wnon wnon 2.242611 1.752643 2.869554\n",
      "wspl wspl 3.063913 2.175813 4.314508\n",
      "wtru wtru 2.458099 1.982799 3.047335\n",
      "wall wall 1.454474 1.266626 1.670181\n",
      "[1] 0.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.61).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.356598 1.178615 1.561458\n",
      "wnon wnon 2.229017 1.742512 2.851353\n",
      "wspl wspl 3.060429 2.173534 4.309216\n",
      "wtru wtru 2.446199 1.973628 3.031925\n",
      "wall wall 1.452739 1.265411 1.667799\n",
      "[1] 0.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.358076 1.180262 1.562680\n",
      "wnon wnon 2.225696 1.740219 2.846609\n",
      "wspl wspl 3.103864 2.210076 4.359113\n",
      "wtru wtru 2.455618 1.982205 3.042097\n",
      "wall wall 1.454431 1.267256 1.669251\n",
      "[1] 0.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.359289 1.181342 1.564039\n",
      "wnon wnon 2.223495 1.738595 2.843635\n",
      "wspl wspl 3.105316 2.211131 4.361111\n",
      "wtru wtru 2.454075 1.981025 3.040086\n",
      "wall wall 1.455422 1.268142 1.670360\n",
      "[1] 0.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.363412 1.185010 1.568671\n",
      "wnon wnon 2.228237 1.742285 2.849729\n",
      "wspl wspl 3.101339 2.208604 4.354924\n",
      "wtru wtru 2.457157 1.983548 3.043848\n",
      "wall wall 1.459297 1.271590 1.674713\n",
      "[1] 0.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.364528 1.186007 1.569920\n",
      "wnon wnon 2.227605 1.741849 2.848826\n",
      "wspl wspl 3.102242 2.209288 4.356113\n",
      "wtru wtru 2.456802 1.983305 3.043342\n",
      "wall wall 1.460274 1.272463 1.675805\n",
      "[1] 0.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.366405 1.187668 1.572040\n",
      "wnon wnon 2.227840 1.742089 2.849035\n",
      "wspl wspl 3.091468 2.201987 4.340249\n",
      "wtru wtru 2.454734 1.981729 3.040638\n",
      "wall wall 1.461939 1.273942 1.677678\n",
      "[1] 1.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, TP53, NOTCH2.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.373129 1.193652 1.579591\n",
      "wnon wnon 2.226893 1.741530 2.847525\n",
      "wspl wspl 3.091958 2.202648 4.340324\n",
      "wtru wtru 2.453964 1.981219 3.039512\n",
      "wall wall 1.467879 1.279234 1.684343\n",
      "[1] 1.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.375952 1.196152 1.582779\n",
      "wnon wnon 2.246950 1.758378 2.871273\n",
      "wspl wspl 3.159962 2.256969 4.424235\n",
      "wtru wtru 2.486836 2.009310 3.077849\n",
      "wall wall 1.473444 1.284162 1.690626\n",
      "[1] 1.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.376793 1.196905 1.583719\n",
      "wnon wnon 2.247879 1.759104 2.872463\n",
      "wspl wspl 3.156538 2.254694 4.419106\n",
      "wtru wtru 2.486915 2.009400 3.077908\n",
      "wall wall 1.474197 1.284836 1.691465\n",
      "[1] 1.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.379614 1.199383 1.586929\n",
      "wnon wnon 2.251241 1.761707 2.876803\n",
      "wspl wspl 3.161421 2.258175 4.425955\n",
      "wtru wtru 2.490685 2.012417 3.082617\n",
      "wall wall 1.476991 1.287292 1.694644\n",
      "[1] 1.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.379614 1.199383 1.586929\n",
      "wnon wnon 2.251241 1.761707 2.876803\n",
      "wspl wspl 3.161421 2.258175 4.425955\n",
      "wtru wtru 2.490685 2.012417 3.082617\n",
      "wall wall 1.476991 1.287292 1.694644\n",
      "[1] 1.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.380420 1.200103 1.587829\n",
      "wnon wnon 2.248308 1.759505 2.872904\n",
      "wspl wspl 3.161245 2.258103 4.425605\n",
      "wtru wtru 2.488210 2.010478 3.079462\n",
      "wall wall 1.477535 1.287780 1.695250\n",
      "[1] 1.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.380420 1.200103 1.587829\n",
      "wnon wnon 2.248308 1.759505 2.872904\n",
      "wspl wspl 3.161245 2.258103 4.425605\n",
      "wtru wtru 2.488210 2.010478 3.079462\n",
      "wall wall 1.477535 1.287780 1.695250\n",
      "[1] 1.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.380420 1.200103 1.587829\n",
      "wnon wnon 2.248308 1.759505 2.872904\n",
      "wspl wspl 3.161245 2.258103 4.425605\n",
      "wtru wtru 2.488210 2.010478 3.079462\n",
      "wall wall 1.477535 1.287780 1.695250\n",
      "[1] 1.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.380420 1.200103 1.587829\n",
      "wnon wnon 2.248308 1.759505 2.872904\n",
      "wspl wspl 3.161245 2.258103 4.425605\n",
      "wtru wtru 2.488210 2.010478 3.079462\n",
      "wall wall 1.477535 1.287780 1.695250\n",
      "[1] 1.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.380420 1.200103 1.587829\n",
      "wnon wnon 2.248308 1.759505 2.872904\n",
      "wspl wspl 3.161245 2.258103 4.425605\n",
      "wtru wtru 2.488210 2.010478 3.079462\n",
      "wall wall 1.477535 1.287780 1.695250\n",
      "[1] 1.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 1.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 2.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.382326 1.201793 1.589979\n",
      "wnon wnon 2.250576 1.761268 2.875820\n",
      "wspl wspl 3.164470 2.260412 4.430108\n",
      "wtru wtru 2.490740 2.012512 3.082608\n",
      "wall wall 1.479405 1.289439 1.697359\n",
      "[1] 2.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] 2.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.68).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH1, FAT1, NOTCH2, TP53.”    Regression model for indels (theta = 1.19)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.383440 1.202788 1.591226\n",
      "wnon wnon 2.249944 1.760832 2.874918\n",
      "wspl wspl 3.165376 2.261099 4.431297\n",
      "wtru wtru 2.490382 2.012265 3.082098\n",
      "wall wall 1.480381 1.290310 1.698449\n",
      "[1] \"PD20399\"\n",
      "[1] 1\n",
      "[1] 0.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "Warning message:\n",
      "“glm.fit: fitted rates numerically 0 occurred”[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.78).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name          mle     cilow   cihigh\n",
      "wmis wmis 1.159052e+00 0.6351110 2.115225\n",
      "wnon wnon 9.917755e-01 0.2124781 4.629270\n",
      "wspl wspl 5.497088e-08 0.0000000      Inf\n",
      "wtru wtru 6.538468e-01 0.1428983 2.991747\n",
      "wall wall 1.130038e+00 0.6221841 2.052423\n",
      "[1] 0.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.05).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow   cihigh\n",
      "wmis wmis 1.254616 0.9790492 1.607746\n",
      "wnon wnon 2.030307 1.2549954 3.284591\n",
      "wspl wspl 2.216889 1.1174294 4.398128\n",
      "wtru wtru 2.085104 1.3770377 3.157256\n",
      "wall wall 1.321637 1.0356815 1.686546\n",
      "[1] 0.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.94).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.45e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.232887 0.999590 1.520633\n",
      "wnon wnon 1.988293 1.335478 2.960221\n",
      "wspl wspl 2.343510 1.343843 4.086818\n",
      "wtru wtru 2.090491 1.483758 2.945327\n",
      "wall wall 1.303323 1.060566 1.601647\n",
      "[1] 0.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.53).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.268770 1.041962 1.544949\n",
      "wnon wnon 2.005100 1.388222 2.896098\n",
      "wspl wspl 2.451414 1.465517 4.100553\n",
      "wtru wtru 2.130218 1.550843 2.926039\n",
      "wall wall 1.341423 1.105420 1.627811\n",
      "[1] 0.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.45).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.236753 1.021751 1.496996\n",
      "wnon wnon 2.055998 1.447270 2.920759\n",
      "wspl wspl 2.327956 1.410170 3.843067\n",
      "wtru wtru 2.133024 1.572115 2.894055\n",
      "wall wall 1.313468 1.088878 1.584380\n",
      "[1] 0.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.53).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.247569 1.034142 1.505045\n",
      "wnon wnon 2.107530 1.492552 2.975899\n",
      "wspl wspl 2.283330 1.386323 3.760737\n",
      "wtru wtru 2.157604 1.597205 2.914625\n",
      "wall wall 1.324874 1.101889 1.592984\n",
      "[1] 0.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.45).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.251066 1.038857 1.506622\n",
      "wnon wnon 2.117026 1.507264 2.973466\n",
      "wspl wspl 2.357179 1.444429 3.846707\n",
      "wtru wtru 2.184601 1.624633 2.937574\n",
      "wall wall 1.330701 1.108697 1.597157\n",
      "[1] 0.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.45).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.259306 1.045990 1.516126\n",
      "wnon wnon 2.196285 1.569893 3.072609\n",
      "wspl wspl 2.362134 1.447895 3.853649\n",
      "wtru wtru 2.242959 1.671883 3.009102\n",
      "wall wall 1.342988 1.119227 1.611485\n",
      "[1] 0.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.271363 1.056876 1.529378\n",
      "wnon wnon 2.249567 1.611479 3.140315\n",
      "wspl wspl 2.378269 1.458107 3.879113\n",
      "wtru wtru 2.285772 1.706148 3.062310\n",
      "wall wall 1.356920 1.131726 1.626924\n",
      "[1] 0.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.272558 1.058516 1.529882\n",
      "wnon wnon 2.245902 1.609269 3.134389\n",
      "wspl wspl 2.348110 1.440732 3.826959\n",
      "wtru wtru 2.274899 1.698730 3.046490\n",
      "wall wall 1.357142 1.132564 1.626253\n",
      "[1] 0.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.43).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.279270 1.064221 1.537775\n",
      "wnon wnon 2.231291 1.598836 3.113929\n",
      "wspl wspl 2.356544 1.445905 3.840710\n",
      "wtru wtru 2.266611 1.692523 3.035426\n",
      "wall wall 1.362659 1.137267 1.632721\n",
      "[1] 0.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277052 1.062913 1.534332\n",
      "wnon wnon 2.210718 1.584768 3.083907\n",
      "wspl wspl 2.345978 1.439883 3.822265\n",
      "wtru wtru 2.248736 1.679846 3.010284\n",
      "wall wall 1.359392 1.135111 1.627989\n",
      "[1] 0.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277052 1.062913 1.534332\n",
      "wnon wnon 2.210718 1.584768 3.083907\n",
      "wspl wspl 2.345978 1.439883 3.822265\n",
      "wtru wtru 2.248736 1.679846 3.010284\n",
      "wall wall 1.359392 1.135111 1.627989\n",
      "[1] 0.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277052 1.062913 1.534332\n",
      "wnon wnon 2.210718 1.584768 3.083907\n",
      "wspl wspl 2.345978 1.439883 3.822265\n",
      "wtru wtru 2.248736 1.679846 3.010284\n",
      "wall wall 1.359392 1.135111 1.627989\n",
      "[1] 0.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.280179 1.065571 1.538009\n",
      "wnon wnon 2.186658 1.567411 3.050553\n",
      "wspl wspl 2.348675 1.441564 3.826588\n",
      "wtru wtru 2.231895 1.667171 2.987910\n",
      "wall wall 1.361195 1.136668 1.630074\n",
      "[1] 0.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.280179 1.065571 1.538009\n",
      "wnon wnon 2.186658 1.567411 3.050553\n",
      "wspl wspl 2.348675 1.441564 3.826588\n",
      "wtru wtru 2.231895 1.667171 2.987910\n",
      "wall wall 1.361195 1.136668 1.630074\n",
      "[1] 0.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.44).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.280179 1.065571 1.538009\n",
      "wnon wnon 2.186658 1.567411 3.050553\n",
      "wspl wspl 2.348675 1.441564 3.826588\n",
      "wtru wtru 2.231895 1.667171 2.987910\n",
      "wall wall 1.361195 1.136668 1.630074\n",
      "[1] 0.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.38).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.271590 1.058847 1.527079\n",
      "wnon wnon 2.176190 1.560230 3.035325\n",
      "wspl wspl 2.336367 1.434209 3.806005\n",
      "wtru wtru 2.220924 1.659360 2.972535\n",
      "wall wall 1.352257 1.129661 1.618715\n",
      "[1] 0.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.38).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276260 1.062793 1.532604\n",
      "wnon wnon 2.181879 1.564252 3.043368\n",
      "wspl wspl 2.342795 1.438149 3.816497\n",
      "wtru wtru 2.226820 1.663710 2.980524\n",
      "wall wall 1.356879 1.133568 1.624182\n",
      "[1] 0.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.38).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276260 1.062793 1.532604\n",
      "wnon wnon 2.181879 1.564252 3.043368\n",
      "wspl wspl 2.342795 1.438149 3.816497\n",
      "wtru wtru 2.226820 1.663710 2.980524\n",
      "wall wall 1.356879 1.133568 1.624182\n",
      "[1] 1.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.38).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276260 1.062793 1.532604\n",
      "wnon wnon 2.181879 1.564252 3.043368\n",
      "wspl wspl 2.342795 1.438149 3.816497\n",
      "wtru wtru 2.226820 1.663710 2.980524\n",
      "wall wall 1.356879 1.133568 1.624182\n",
      "[1] 1.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.38).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276260 1.062793 1.532604\n",
      "wnon wnon 2.181879 1.564252 3.043368\n",
      "wspl wspl 2.342795 1.438149 3.816497\n",
      "wtru wtru 2.226820 1.663710 2.980524\n",
      "wall wall 1.356879 1.133568 1.624182\n",
      "[1] 1.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277475 1.063816 1.534046\n",
      "wnon wnon 2.219794 1.594539 3.090227\n",
      "wspl wspl 2.345851 1.440025 3.821471\n",
      "wtru wtru 2.254963 1.686706 3.014667\n",
      "wall wall 1.360746 1.136857 1.628728\n",
      "[1] 1.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277475 1.063816 1.534046\n",
      "wnon wnon 2.219794 1.594539 3.090227\n",
      "wspl wspl 2.345851 1.440025 3.821471\n",
      "wtru wtru 2.254963 1.686706 3.014667\n",
      "wall wall 1.360746 1.136857 1.628728\n",
      "[1] 1.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277475 1.063816 1.534046\n",
      "wnon wnon 2.219794 1.594539 3.090227\n",
      "wspl wspl 2.345851 1.440025 3.821471\n",
      "wtru wtru 2.254963 1.686706 3.014667\n",
      "wall wall 1.360746 1.136857 1.628728\n",
      "[1] 1.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.277475 1.063816 1.534046\n",
      "wnon wnon 2.219794 1.594539 3.090227\n",
      "wspl wspl 2.345851 1.440025 3.821471\n",
      "wtru wtru 2.254963 1.686706 3.014667\n",
      "wall wall 1.360746 1.136857 1.628728\n",
      "[1] 1.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 1.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] 2.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.39).\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“    Genes were excluded from the indel background model based on the substitution data: NOTCH2, NOTCH1.”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for indels (theta = 6.78e+03)\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow   cihigh\n",
      "wmis wmis 1.276326 1.062856 1.532669\n",
      "wnon wnon 2.219666 1.594449 3.090042\n",
      "wspl wspl 2.429065 1.502681 3.926552\n",
      "wtru wtru 2.278386 1.705999 3.042817\n",
      "wall wall 1.361287 1.137328 1.629346\n",
      "[1] \"PD21910\"\n",
      "[1] 1\n",
      "[1] 0.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "[3] Estimating global rates...\n",
      "Warning message:\n",
      "“glm.fit: fitted rates numerically 0 occurred”Warning message:\n",
      "“glm.fit: fitted rates numerically 0 occurred”[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”Warning message in theta.ml(Y, mu, sum(w), w, limit = control$maxit, trace = control$trace > :\n",
      "“iteration limit reached”    Regression model for substitutions (theta = 5.17e+03).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name          mle     cilow   cihigh\n",
      "wmis wmis 2.644038e+00 0.2936180 23.80963\n",
      "wnon wnon 2.069169e-09 0.0000000      Inf\n",
      "wspl wspl 4.993680e-07 0.0000000      Inf\n",
      "wtru wtru 2.775044e-08 0.0000000      Inf\n",
      "wall wall 2.156958e+00 0.2441342 19.05701\n",
      "[1] 0.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 9.25).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name       mle     cilow    cihigh\n",
      "wmis wmis 1.8576816 0.9908212  3.482950\n",
      "wnon wnon 0.5228357 0.0655607  4.169527\n",
      "wspl wspl 3.3091775 0.3785006 28.931671\n",
      "wtru wtru 0.9194588 0.1986686  4.255351\n",
      "wall wall 1.7644880 0.9458235  3.291753\n",
      "[1] 0.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.95).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.605220 0.9628757  2.676077\n",
      "wnon wnon 1.470919 0.4727855  4.576288\n",
      "wspl wspl 3.972226 0.8313693 18.979028\n",
      "wtru wtru 1.900794 0.7257828  4.978101\n",
      "wall wall 1.621917 0.9789037  2.687305\n",
      "[1] 0.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 1.57).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.459440 0.9289243  2.292937\n",
      "wnon wnon 1.504260 0.5827637  3.882876\n",
      "wspl wspl 3.706801 0.9987026 13.758222\n",
      "wtru wtru 1.904123 0.8460739  4.285304\n",
      "wall wall 1.486845 0.9517024  2.322898\n",
      "[1] 0.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 4.66).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.390054 0.9104731  2.122249\n",
      "wnon wnon 1.591364 0.6875186  3.683448\n",
      "wspl wspl 4.261582 1.3480960 13.471652\n",
      "wtru wtru 2.056281 0.9997284  4.229439\n",
      "wall wall 1.436904 0.9470128  2.180216\n",
      "[1] 0.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.27).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.466763 0.9704238  2.216962\n",
      "wnon wnon 1.624735 0.7041389  3.748924\n",
      "wspl wspl 5.914425 2.2119110 15.814573\n",
      "wtru wtru 2.449919 1.2384094  4.846623\n",
      "wall wall 1.535450 1.0221295  2.306563\n",
      "[1] 0.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.23).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.507410 0.9996618  2.273053\n",
      "wnon wnon 1.733793 0.7787110  3.860274\n",
      "wspl wspl 5.997719 2.2463590 16.013753\n",
      "wtru wtru 2.514749 1.2907884  4.899304\n",
      "wall wall 1.580416 1.0541942  2.369310\n",
      "[1] 0.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 4.45).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.502630 1.0087578  2.238293\n",
      "wnon wnon 1.611855 0.7313756  3.552315\n",
      "wspl wspl 5.469263 2.0630520 14.499314\n",
      "wtru wtru 2.316624 1.2013176  4.467382\n",
      "wall wall 1.559601 1.0526542  2.310687\n",
      "[1] 0.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 8.14).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.530314 1.0338850  2.265107\n",
      "wnon wnon 1.615661 0.7347939  3.552505\n",
      "wspl wspl 5.013043 1.8956817 13.256762\n",
      "wtru wtru 2.273496 1.1822170  4.372110\n",
      "wall wall 1.583213 1.0751490  2.331363\n",
      "[1] 0.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 8.14).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.530314 1.0338850  2.265107\n",
      "wnon wnon 1.615661 0.7347939  3.552505\n",
      "wspl wspl 5.013043 1.8956817 13.256762\n",
      "wtru wtru 2.273496 1.1822170  4.372110\n",
      "wall wall 1.583213 1.0751490  2.331363\n",
      "[1] 0.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 8.48).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.590491 1.0759525  2.351090\n",
      "wnon wnon 1.660156 0.7546211  3.652319\n",
      "wspl wspl 4.767315 1.8135640 12.531841\n",
      "wtru wtru 2.301423 1.1979150  4.421470\n",
      "wall wall 1.642723 1.1168003  2.416313\n",
      "[1] 0.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 4.42).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.549350 1.0529423  2.279788\n",
      "wnon wnon 1.616888 0.7368501  3.547977\n",
      "wspl wspl 4.658169 1.7757678 12.219242\n",
      "wtru wtru 2.242114 1.1703517  4.295356\n",
      "wall wall 1.599934 1.0926810  2.342669\n",
      "[1] 0.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 3.51).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.515283 1.0345412  2.219422\n",
      "wnon wnon 1.564298 0.7150959  3.421957\n",
      "wspl wspl 4.276659 1.6380338 11.165713\n",
      "wtru wtru 2.141436 1.1218778  4.087566\n",
      "wall wall 1.559605 1.0698056  2.273653\n",
      "[1] 0.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.38).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.474692 1.0110252  2.151002\n",
      "wnon wnon 1.522859 0.6978222  3.323339\n",
      "wspl wspl 4.176197 1.6025871 10.882792\n",
      "wtru wtru 2.085339 1.0953497  3.970091\n",
      "wall wall 1.517713 1.0453924  2.203435\n",
      "[1] 0.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.38).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle    cilow    cihigh\n",
      "wmis wmis 1.481497 1.016242  2.159756\n",
      "wnon wnon 1.673281 0.788208  3.552195\n",
      "wspl wspl 4.193686 1.609896 10.924310\n",
      "wtru wtru 2.205254 1.172683  4.147025\n",
      "wall wall 1.533963 1.057274  2.225577\n",
      "[1] 0.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 0.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 0.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 0.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 0.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.53\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.58\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.63\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.68\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.73\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.78\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.83\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.88\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.93\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 1.98\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.03\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.08\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.13\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.18\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.38\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.43\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n",
      "[1] 2.48\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[1] Loading the environment...\n",
      "[2] Annotating the mutations...\n",
      "Warning message in dndscv(x1, gene_list = target_genes, outp = 3, max_muts_per_gene_per_sample = Inf, :\n",
      "“Same mutations observed in different sampleIDs. Please verify that these are independent events and remove duplicates otherwise.”[3] Estimating global rates...\n",
      "[4] Running dNdSloc...\n",
      "[5] Running dNdScv...\n",
      "    Regression model for substitutions (theta = 2.74).\n",
      "Warning message:\n",
      "“Unknown or uninitialised column: 'donor'.”"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     name      mle     cilow    cihigh\n",
      "wmis wmis 1.458360 1.0045762  2.117126\n",
      "wnon wnon 1.653544 0.7801215  3.504847\n",
      "wspl wspl 4.101421 1.5791793 10.652147\n",
      "wtru wtru 2.174981 1.1590106  4.081534\n",
      "wall wall 1.509772 1.0449248  2.181413\n"
     ]
    }
   ],
   "source": [
    "\n",
    "dfdnds.patient <- data.frame()\n",
    "dfdnds.genes.patient <- data.frame()\n",
    "i <- 1\n",
    "for (p in unique(df$patient)){\n",
    "  print(p)\n",
    "  print(i)\n",
    "  for (cutoff in areacutoff){\n",
    "    print(cutoff)\n",
    "    x1 <- df %>%\n",
    "      filter(patient == p) %>%\n",
    "      filter(clone.area < cutoff) #filter for mutations with ccf < cutoff\n",
    "    x <- dndscv(x1, gene_list = target_genes,\n",
    "                outp = 3, max_muts_per_gene_per_sample = Inf, \n",
    "                max_coding_muts_per_sample = Inf)\n",
    "    out <- x$globaldnds %>%\n",
    "        mutate(areacutoff = cutoff, nmutations = length(x1$donor), patient = p)\n",
    "    dfdnds.patient <- rbind(dfdnds.patient, out)\n",
    "    combined <- left_join(x$sel_cv, x$sel_loc, by = c(\"gene_name\", \"n_syn\", \"n_mis\", \"n_non\", \"n_spl\"))\n",
    "    out2 <- combined %>%\n",
    "      mutate(areacutoff = cutoff, patient = p)\n",
    "    dfdnds.genes.patient <- bind_rows(dfdnds.genes.patient, out2)\n",
    "    print(x$globaldnds)\n",
    "  }\n",
    "  i + i +1\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "write_csv(dfdnds.patient, \"data/skin/dnds_patient_combined.csv\")\n",
    "write_csv(dfdnds.genes.patient, \"data/skin/dnds_patient_genes_combined.csv\")\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.5.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
